// file generated from a CSV file using the script: observatory-misc/EDAM.py
export const EDAMreversed = {
	format: {
		SMILES: 'http://edamontology.org/format_1196',
		InChI: 'http://edamontology.org/format_1197',
		mf: 'http://edamontology.org/format_1198',
		InChIKey: 'http://edamontology.org/format_1199',
		smarts: 'http://edamontology.org/format_1200',
		'unambiguous pure': 'http://edamontology.org/format_1206',
		nucleotide: 'http://edamontology.org/format_1207',
		protein: 'http://edamontology.org/format_1208',
		consensus: 'http://edamontology.org/format_1209',
		'pure nucleotide': 'http://edamontology.org/format_1210',
		'unambiguous pure nucleotide': 'http://edamontology.org/format_1211',
		dna: 'http://edamontology.org/format_1212',
		rna: 'http://edamontology.org/format_1213',
		'unambiguous pure dna': 'http://edamontology.org/format_1214',
		'pure dna': 'http://edamontology.org/format_1215',
		'unambiguous pure rna sequence': 'http://edamontology.org/format_1216',
		'pure rna': 'http://edamontology.org/format_1217',
		'unambiguous pure protein': 'http://edamontology.org/format_1218',
		'pure protein': 'http://edamontology.org/format_1219',
		'UniGene entry format': 'http://edamontology.org/format_1228',
		'COG sequence cluster format': 'http://edamontology.org/format_1247',
		'EMBL feature location': 'http://edamontology.org/format_1248',
		quicktandem: 'http://edamontology.org/format_1295',
		'Sanger inverted repeats': 'http://edamontology.org/format_1296',
		'EMBOSS repeat': 'http://edamontology.org/format_1297',
		'est2genome format': 'http://edamontology.org/format_1316',
		'restrict format': 'http://edamontology.org/format_1318',
		'restover format': 'http://edamontology.org/format_1319',
		'REBASE restriction sites': 'http://edamontology.org/format_1320',
		'FASTA search results format': 'http://edamontology.org/format_1332',
		'BLAST results': 'http://edamontology.org/format_1333',
		mspcrunch: 'http://edamontology.org/format_1334',
		'Smith-Waterman format': 'http://edamontology.org/format_1335',
		dhf: 'http://edamontology.org/format_1336',
		lhf: 'http://edamontology.org/format_1337',
		'InterPro hits format': 'http://edamontology.org/format_1341',
		'InterPro protein view report format':
			'http://edamontology.org/format_1342',
		'InterPro match table format': 'http://edamontology.org/format_1343',
		'HMMER Dirichlet prior': 'http://edamontology.org/format_1349',
		'MEME Dirichlet prior': 'http://edamontology.org/format_1350',
		'HMMER emission and transition': 'http://edamontology.org/format_1351',
		'prosite-pattern': 'http://edamontology.org/format_1356',
		'EMBOSS sequence pattern': 'http://edamontology.org/format_1357',
		'meme-motif': 'http://edamontology.org/format_1360',
		'prosite-profile': 'http://edamontology.org/format_1366',
		'JASPAR format': 'http://edamontology.org/format_1367',
		'MEME background Markov model': 'http://edamontology.org/format_1369',
		'HMMER format': 'http://edamontology.org/format_1370',
		'HMMER-aln': 'http://edamontology.org/format_1391',
		'DIALIGN format': 'http://edamontology.org/format_1392',
		daf: 'http://edamontology.org/format_1393',
		'Sequence-MEME profile alignment': 'http://edamontology.org/format_1419',
		'HMMER profile alignment (sequences versus HMMs)':
			'http://edamontology.org/format_1421',
		'HMMER profile alignment (HMM versus sequences)':
			'http://edamontology.org/format_1422',
		'Phylip distance matrix': 'http://edamontology.org/format_1423',
		'ClustalW dendrogram': 'http://edamontology.org/format_1424',
		'Phylip tree raw': 'http://edamontology.org/format_1425',
		'Phylip continuous quantitative characters':
			'http://edamontology.org/format_1430',
		'Phylogenetic property values format':
			'http://edamontology.org/format_1431',
		'Phylip character frequencies format':
			'http://edamontology.org/format_1432',
		'Phylip discrete states format': 'http://edamontology.org/format_1433',
		'Phylip cliques format': 'http://edamontology.org/format_1434',
		'Phylip tree format': 'http://edamontology.org/format_1435',
		'TreeBASE format': 'http://edamontology.org/format_1436',
		'TreeFam format': 'http://edamontology.org/format_1437',
		'Phylip tree distance format': 'http://edamontology.org/format_1445',
		dssp: 'http://edamontology.org/format_1454',
		hssp: 'http://edamontology.org/format_1455',
		'Dot-bracket format': 'http://edamontology.org/format_1457',
		'Vienna local RNA secondary structure format':
			'http://edamontology.org/format_1458',
		'PDB database entry format': 'http://edamontology.org/format_1475',
		PDB: 'http://edamontology.org/format_1476',
		mmCIF: 'http://edamontology.org/format_1477',
		PDBML: 'http://edamontology.org/format_1478',
		'Domainatrix 3D-1D scoring matrix format':
			'http://edamontology.org/format_1500',
		aaindex: 'http://edamontology.org/format_1504',
		'IntEnz enzyme report format': 'http://edamontology.org/format_1511',
		'BRENDA enzyme report format': 'http://edamontology.org/format_1512',
		'KEGG REACTION enzyme report format': 'http://edamontology.org/format_1513',
		'KEGG ENZYME enzyme report format': 'http://edamontology.org/format_1514',
		'REBASE proto enzyme report format': 'http://edamontology.org/format_1515',
		'REBASE withrefm enzyme report format':
			'http://edamontology.org/format_1516',
		'Pcons report format': 'http://edamontology.org/format_1551',
		'ProQ report format': 'http://edamontology.org/format_1552',
		'SMART domain assignment report format':
			'http://edamontology.org/format_1563',
		'BIND entry format': 'http://edamontology.org/format_1568',
		'IntAct entry format': 'http://edamontology.org/format_1569',
		'InterPro entry format': 'http://edamontology.org/format_1570',
		'InterPro entry abstract format': 'http://edamontology.org/format_1571',
		'Gene3D entry format': 'http://edamontology.org/format_1572',
		'PIRSF entry format': 'http://edamontology.org/format_1573',
		'PRINTS entry format': 'http://edamontology.org/format_1574',
		'Panther Families and HMMs entry format':
			'http://edamontology.org/format_1575',
		'Pfam entry format': 'http://edamontology.org/format_1576',
		'SMART entry format': 'http://edamontology.org/format_1577',
		'Superfamily entry format': 'http://edamontology.org/format_1578',
		'TIGRFam entry format': 'http://edamontology.org/format_1579',
		'ProDom entry format': 'http://edamontology.org/format_1580',
		'FSSP entry format': 'http://edamontology.org/format_1581',
		findkm: 'http://edamontology.org/format_1582',
		'Ensembl gene report format': 'http://edamontology.org/format_1603',
		'DictyBase gene report format': 'http://edamontology.org/format_1604',
		'CGD gene report format': 'http://edamontology.org/format_1605',
		'DragonDB gene report format': 'http://edamontology.org/format_1606',
		'EcoCyc gene report format': 'http://edamontology.org/format_1607',
		'FlyBase gene report format': 'http://edamontology.org/format_1608',
		'Gramene gene report format': 'http://edamontology.org/format_1609',
		'KEGG GENES gene report format': 'http://edamontology.org/format_1610',
		'MaizeGDB gene report format': 'http://edamontology.org/format_1611',
		'MGD gene report format': 'http://edamontology.org/format_1612',
		'RGD gene report format': 'http://edamontology.org/format_1613',
		'SGD gene report format': 'http://edamontology.org/format_1614',
		'GeneDB gene report format': 'http://edamontology.org/format_1615',
		'TAIR gene report format': 'http://edamontology.org/format_1616',
		'WormBase gene report format': 'http://edamontology.org/format_1617',
		'ZFIN gene report format': 'http://edamontology.org/format_1618',
		'TIGR gene report format': 'http://edamontology.org/format_1619',
		'dbSNP polymorphism report format': 'http://edamontology.org/format_1620',
		'OMIM entry format': 'http://edamontology.org/format_1623',
		'HGVbase entry format': 'http://edamontology.org/format_1624',
		'HIVDB entry format': 'http://edamontology.org/format_1625',
		'KEGG DISEASE entry format': 'http://edamontology.org/format_1626',
		'Primer3 primer': 'http://edamontology.org/format_1627',
		ABI: 'http://edamontology.org/format_1628',
		mira: 'http://edamontology.org/format_1629',
		CAF: 'http://edamontology.org/format_1630',
		EXP: 'http://edamontology.org/format_1631',
		SCF: 'http://edamontology.org/format_1632',
		PHD: 'http://edamontology.org/format_1633',
		dat: 'http://edamontology.org/format_1637',
		cel: 'http://edamontology.org/format_1638',
		affymetrix: 'http://edamontology.org/format_1639',
		'ArrayExpress entry format': 'http://edamontology.org/format_1640',
		'affymetrix-exp': 'http://edamontology.org/format_1641',
		CHP: 'http://edamontology.org/format_1644',
		'EMDB entry format': 'http://edamontology.org/format_1645',
		'KEGG PATHWAY entry format': 'http://edamontology.org/format_1647',
		'MetaCyc entry format': 'http://edamontology.org/format_1648',
		'HumanCyc entry format': 'http://edamontology.org/format_1649',
		'INOH entry format': 'http://edamontology.org/format_1650',
		'PATIKA entry format': 'http://edamontology.org/format_1651',
		'Reactome entry format': 'http://edamontology.org/format_1652',
		'aMAZE entry format': 'http://edamontology.org/format_1653',
		'CPDB entry format': 'http://edamontology.org/format_1654',
		'Panther Pathways entry format': 'http://edamontology.org/format_1655',
		'Taverna workflow format': 'http://edamontology.org/format_1665',
		'BioModel mathematical model format': 'http://edamontology.org/format_1666',
		'KEGG LIGAND entry format': 'http://edamontology.org/format_1697',
		'KEGG COMPOUND entry format': 'http://edamontology.org/format_1698',
		'KEGG PLANT entry format': 'http://edamontology.org/format_1699',
		'KEGG GLYCAN entry format': 'http://edamontology.org/format_1700',
		'PubChem entry format': 'http://edamontology.org/format_1701',
		'ChemSpider entry format': 'http://edamontology.org/format_1702',
		'ChEBI entry format': 'http://edamontology.org/format_1703',
		'MSDchem ligand dictionary entry format':
			'http://edamontology.org/format_1704',
		'HET group dictionary entry format': 'http://edamontology.org/format_1705',
		'KEGG DRUG entry format': 'http://edamontology.org/format_1706',
		'PubMed citation': 'http://edamontology.org/format_1734',
		'Medline Display Format': 'http://edamontology.org/format_1735',
		'CiteXplore-core': 'http://edamontology.org/format_1736',
		'CiteXplore-all': 'http://edamontology.org/format_1737',
		pmc: 'http://edamontology.org/format_1739',
		'iHOP format': 'http://edamontology.org/format_1740',
		'OSCAR format': 'http://edamontology.org/format_1741',
		'PDB atom record format': 'http://edamontology.org/format_1747',
		'CATH chain report format': 'http://edamontology.org/format_1760',
		'CATH PDB report format': 'http://edamontology.org/format_1761',
		'NCBI gene report format': 'http://edamontology.org/format_1782',
		'GeneIlluminator gene report format': 'http://edamontology.org/format_1808',
		'BacMap gene card format': 'http://edamontology.org/format_1809',
		'ColiCard report format': 'http://edamontology.org/format_1810',
		'PlasMapper TextMap': 'http://edamontology.org/format_1861',
		newick: 'http://edamontology.org/format_1910',
		'TreeCon format': 'http://edamontology.org/format_1911',
		'Nexus format': 'http://edamontology.org/format_1912',
		Format: 'http://edamontology.org/format_1915',
		'Atomic data format': 'http://edamontology.org/format_1918',
		'Sequence record format': 'http://edamontology.org/format_1919',
		'Sequence feature annotation format': 'http://edamontology.org/format_1920',
		'Alignment format': 'http://edamontology.org/format_1921',
		acedb: 'http://edamontology.org/format_1923',
		'clustal sequence format': 'http://edamontology.org/format_1924',
		codata: 'http://edamontology.org/format_1925',
		dbid: 'http://edamontology.org/format_1926',
		'EMBL format': 'http://edamontology.org/format_1927',
		'Staden experiment format': 'http://edamontology.org/format_1928',
		FASTA: 'http://edamontology.org/format_1929',
		FASTQ: 'http://edamontology.org/format_1930',
		'FASTQ-illumina': 'http://edamontology.org/format_1931',
		'FASTQ-sanger': 'http://edamontology.org/format_1932',
		'FASTQ-solexa': 'http://edamontology.org/format_1933',
		'fitch program': 'http://edamontology.org/format_1934',
		GCG: 'http://edamontology.org/format_1935',
		'GenBank format': 'http://edamontology.org/format_1936',
		genpept: 'http://edamontology.org/format_1937',
		'GFF2-seq': 'http://edamontology.org/format_1938',
		'GFF3-seq': 'http://edamontology.org/format_1939',
		'giFASTA format': 'http://edamontology.org/format_1940',
		hennig86: 'http://edamontology.org/format_1941',
		ig: 'http://edamontology.org/format_1942',
		igstrict: 'http://edamontology.org/format_1943',
		jackknifer: 'http://edamontology.org/format_1944',
		'mase format': 'http://edamontology.org/format_1945',
		'mega-seq': 'http://edamontology.org/format_1946',
		'GCG MSF': 'http://edamontology.org/format_1947',
		'nbrf/pir': 'http://edamontology.org/format_1948',
		'nexus-seq': 'http://edamontology.org/format_1949',
		pdbatom: 'http://edamontology.org/format_1950',
		pdbatomnuc: 'http://edamontology.org/format_1951',
		pdbseqresnuc: 'http://edamontology.org/format_1952',
		pdbseqres: 'http://edamontology.org/format_1953',
		'Pearson format': 'http://edamontology.org/format_1954',
		'phylip sequence format': 'http://edamontology.org/format_1955',
		'phylipnon sequence format': 'http://edamontology.org/format_1956',
		raw: 'http://edamontology.org/format_1957',
		refseqp: 'http://edamontology.org/format_1958',
		'selex sequence format': 'http://edamontology.org/format_1959',
		'Staden format': 'http://edamontology.org/format_1960',
		'Stockholm format': 'http://edamontology.org/format_1961',
		'strider format': 'http://edamontology.org/format_1962',
		'UniProtKB format': 'http://edamontology.org/format_1963',
		'plain text format (unformatted)': 'http://edamontology.org/format_1964',
		'treecon sequence format': 'http://edamontology.org/format_1965',
		'ASN.1 sequence format': 'http://edamontology.org/format_1966',
		'DAS format': 'http://edamontology.org/format_1967',
		dasdna: 'http://edamontology.org/format_1968',
		'debug-seq': 'http://edamontology.org/format_1969',
		jackknifernon: 'http://edamontology.org/format_1970',
		'meganon sequence format': 'http://edamontology.org/format_1971',
		'NCBI format': 'http://edamontology.org/format_1972',
		nexusnon: 'http://edamontology.org/format_1973',
		GFF2: 'http://edamontology.org/format_1974',
		GFF3: 'http://edamontology.org/format_1975',
		pir: 'http://edamontology.org/format_1976',
		'swiss feature': 'http://edamontology.org/format_1977',
		DASGFF: 'http://edamontology.org/format_1978',
		'debug-feat': 'http://edamontology.org/format_1979',
		'EMBL feature': 'http://edamontology.org/format_1980',
		'GenBank feature': 'http://edamontology.org/format_1981',
		'ClustalW format': 'http://edamontology.org/format_1982',
		debug: 'http://edamontology.org/format_1983',
		'FASTA-aln': 'http://edamontology.org/format_1984',
		markx0: 'http://edamontology.org/format_1985',
		markx1: 'http://edamontology.org/format_1986',
		markx10: 'http://edamontology.org/format_1987',
		markx2: 'http://edamontology.org/format_1988',
		markx3: 'http://edamontology.org/format_1989',
		match: 'http://edamontology.org/format_1990',
		mega: 'http://edamontology.org/format_1991',
		meganon: 'http://edamontology.org/format_1992',
		'msf alignment format': 'http://edamontology.org/format_1993',
		'nexus alignment format': 'http://edamontology.org/format_1994',
		'nexusnon alignment format': 'http://edamontology.org/format_1995',
		pair: 'http://edamontology.org/format_1996',
		'PHYLIP format': 'http://edamontology.org/format_1997',
		'PHYLIP sequential': 'http://edamontology.org/format_1998',
		'scores format': 'http://edamontology.org/format_1999',
		selex: 'http://edamontology.org/format_2000',
		'EMBOSS simple format': 'http://edamontology.org/format_2001',
		'srs format': 'http://edamontology.org/format_2002',
		srspair: 'http://edamontology.org/format_2003',
		'T-Coffee format': 'http://edamontology.org/format_2004',
		'TreeCon-seq': 'http://edamontology.org/format_2005',
		'Phylogenetic tree format': 'http://edamontology.org/format_2006',
		'Biological pathway or network format':
			'http://edamontology.org/format_2013',
		'Sequence-profile alignment format': 'http://edamontology.org/format_2014',
		'Sequence-profile alignment (HMM) format':
			'http://edamontology.org/format_2015',
		'Amino acid index format': 'http://edamontology.org/format_2017',
		'Article format': 'http://edamontology.org/format_2020',
		'Text mining report format': 'http://edamontology.org/format_2021',
		'Enzyme kinetics report format': 'http://edamontology.org/format_2027',
		'Chemical data format': 'http://edamontology.org/format_2030',
		'Gene annotation format': 'http://edamontology.org/format_2031',
		'Workflow format': 'http://edamontology.org/format_2032',
		'Tertiary structure format': 'http://edamontology.org/format_2033',
		'Biological model format': 'http://edamontology.org/format_2034',
		'Chemical formula format': 'http://edamontology.org/format_2035',
		'Phylogenetic character data format': 'http://edamontology.org/format_2036',
		'Phylogenetic continuous quantitative character format':
			'http://edamontology.org/format_2037',
		'Phylogenetic discrete states format':
			'http://edamontology.org/format_2038',
		'Phylogenetic tree report (cliques) format':
			'http://edamontology.org/format_2039',
		'Phylogenetic tree report (invariants) format':
			'http://edamontology.org/format_2040',
		'Electron microscopy model format': 'http://edamontology.org/format_2045',
		'Phylogenetic tree report (tree distances) format':
			'http://edamontology.org/format_2049',
		'Polymorphism report format': 'http://edamontology.org/format_2051',
		'Protein family report format': 'http://edamontology.org/format_2052',
		'Protein interaction format': 'http://edamontology.org/format_2054',
		'Sequence assembly format': 'http://edamontology.org/format_2055',
		'Microarray experiment data format': 'http://edamontology.org/format_2056',
		'Sequence trace format': 'http://edamontology.org/format_2057',
		'Gene expression report format': 'http://edamontology.org/format_2058',
		'Genotype and phenotype annotation format':
			'http://edamontology.org/format_2059',
		'Map format': 'http://edamontology.org/format_2060',
		'Nucleic acid features (primers) format':
			'http://edamontology.org/format_2061',
		'Protein report format': 'http://edamontology.org/format_2062',
		'Protein report (enzyme) format': 'http://edamontology.org/format_2063',
		'3D-1D scoring matrix format': 'http://edamontology.org/format_2064',
		'Protein structure report (quality evaluation) format':
			'http://edamontology.org/format_2065',
		'Database hits (sequence) format': 'http://edamontology.org/format_2066',
		'Sequence distance matrix format': 'http://edamontology.org/format_2067',
		'Sequence motif format': 'http://edamontology.org/format_2068',
		'Sequence profile format': 'http://edamontology.org/format_2069',
		'Hidden Markov model format': 'http://edamontology.org/format_2072',
		'Dirichlet distribution format': 'http://edamontology.org/format_2074',
		'HMM emission and transition counts format':
			'http://edamontology.org/format_2075',
		'RNA secondary structure format': 'http://edamontology.org/format_2076',
		'Protein secondary structure format': 'http://edamontology.org/format_2077',
		'Sequence range format': 'http://edamontology.org/format_2078',
		pure: 'http://edamontology.org/format_2094',
		unpure: 'http://edamontology.org/format_2095',
		'unambiguous sequence': 'http://edamontology.org/format_2096',
		ambiguous: 'http://edamontology.org/format_2097',
		'Sequence features (repeats) format': 'http://edamontology.org/format_2155',
		'Nucleic acid features (restriction sites) format':
			'http://edamontology.org/format_2158',
		'Gene features (coding region) format':
			'http://edamontology.org/format_2159',
		'Sequence cluster format': 'http://edamontology.org/format_2170',
		'Sequence cluster format (protein)': 'http://edamontology.org/format_2171',
		'Sequence cluster format (nucleic acid)':
			'http://edamontology.org/format_2172',
		'Gene cluster format': 'http://edamontology.org/format_2175',
		'EMBL-like (text)': 'http://edamontology.org/format_2181',
		'FASTQ-like format (text)': 'http://edamontology.org/format_2182',
		EMBLXML: 'http://edamontology.org/format_2183',
		cdsxml: 'http://edamontology.org/format_2184',
		insdxml: 'http://edamontology.org/format_2185',
		geneseq: 'http://edamontology.org/format_2186',
		'UniProt-like (text)': 'http://edamontology.org/format_2187',
		'UniProt format': 'http://edamontology.org/format_2188',
		ipi: 'http://edamontology.org/format_2189',
		medline: 'http://edamontology.org/format_2194',
		'Ontology format': 'http://edamontology.org/format_2195',
		'OBO format': 'http://edamontology.org/format_2196',
		'OWL format': 'http://edamontology.org/format_2197',
		'FASTA-like (text)': 'http://edamontology.org/format_2200',
		'Sequence record full format': 'http://edamontology.org/format_2202',
		'Sequence record lite format': 'http://edamontology.org/format_2203',
		'EMBL format (XML)': 'http://edamontology.org/format_2204',
		'GenBank-like format (text)': 'http://edamontology.org/format_2205',
		'Sequence feature table format (text)':
			'http://edamontology.org/format_2206',
		'Strain data format': 'http://edamontology.org/format_2210',
		'CIP strain data format': 'http://edamontology.org/format_2211',
		'phylip property values': 'http://edamontology.org/format_2243',
		'STRING entry format (HTML)': 'http://edamontology.org/format_2303',
		'STRING entry format (XML)': 'http://edamontology.org/format_2304',
		GFF: 'http://edamontology.org/format_2305',
		GTF: 'http://edamontology.org/format_2306',
		'FASTA-HTML': 'http://edamontology.org/format_2310',
		'EMBL-HTML': 'http://edamontology.org/format_2311',
		'BioCyc enzyme report format': 'http://edamontology.org/format_2322',
		'ENZYME enzyme report format': 'http://edamontology.org/format_2323',
		'PseudoCAP gene report format': 'http://edamontology.org/format_2328',
		'GeneCards gene report format': 'http://edamontology.org/format_2329',
		'Textual format': 'http://edamontology.org/format_2330',
		HTML: 'http://edamontology.org/format_2331',
		XML: 'http://edamontology.org/format_2332',
		'Binary format': 'http://edamontology.org/format_2333',
		'URI format': 'http://edamontology.org/format_2334',
		'NCI-Nature pathway entry format': 'http://edamontology.org/format_2341',
		'Format (by type of data)': 'http://edamontology.org/format_2350',
		'BioXSD (XML)': 'http://edamontology.org/format_2352',
		'RDF format': 'http://edamontology.org/format_2376',
		'GenBank-HTML': 'http://edamontology.org/format_2532',
		'Protein features (domains) format': 'http://edamontology.org/format_2542',
		'EMBL-like format': 'http://edamontology.org/format_2543',
		'FASTQ-like format': 'http://edamontology.org/format_2545',
		'FASTA-like': 'http://edamontology.org/format_2546',
		'uniprotkb-like format': 'http://edamontology.org/format_2547',
		'Sequence feature table format': 'http://edamontology.org/format_2548',
		OBO: 'http://edamontology.org/format_2549',
		'OBO-XML': 'http://edamontology.org/format_2550',
		'Sequence record format (text)': 'http://edamontology.org/format_2551',
		'Sequence record format (XML)': 'http://edamontology.org/format_2552',
		'Sequence feature table format (XML)':
			'http://edamontology.org/format_2553',
		'Alignment format (text)': 'http://edamontology.org/format_2554',
		'Alignment format (XML)': 'http://edamontology.org/format_2555',
		'Phylogenetic tree format (text)': 'http://edamontology.org/format_2556',
		'Phylogenetic tree format (XML)': 'http://edamontology.org/format_2557',
		'EMBL-like (XML)': 'http://edamontology.org/format_2558',
		'GenBank-like format': 'http://edamontology.org/format_2559',
		'STRING entry format': 'http://edamontology.org/format_2560',
		'Sequence assembly format (text)': 'http://edamontology.org/format_2561',
		'Amino acid identifier format': 'http://edamontology.org/format_2562',
		'completely unambiguous': 'http://edamontology.org/format_2566',
		'completely unambiguous pure': 'http://edamontology.org/format_2567',
		'completely unambiguous pure nucleotide':
			'http://edamontology.org/format_2568',
		'completely unambiguous pure dna': 'http://edamontology.org/format_2569',
		'completely unambiguous pure rna sequence':
			'http://edamontology.org/format_2570',
		'Raw sequence format': 'http://edamontology.org/format_2571',
		BAM: 'http://edamontology.org/format_2572',
		SAM: 'http://edamontology.org/format_2573',
		SBML: 'http://edamontology.org/format_2585',
		'completely unambiguous pure protein':
			'http://edamontology.org/format_2607',
		'Bibliographic reference format': 'http://edamontology.org/format_2848',
		'Sequence annotation track format': 'http://edamontology.org/format_2919',
		'Alignment format (pair only)': 'http://edamontology.org/format_2920',
		'Sequence variation annotation format':
			'http://edamontology.org/format_2921',
		'markx0 variant': 'http://edamontology.org/format_2922',
		'mega variant': 'http://edamontology.org/format_2923',
		'Phylip format variant': 'http://edamontology.org/format_2924',
		AB1: 'http://edamontology.org/format_3000',
		ACE: 'http://edamontology.org/format_3001',
		BED: 'http://edamontology.org/format_3003',
		bigBed: 'http://edamontology.org/format_3004',
		WIG: 'http://edamontology.org/format_3005',
		bigWig: 'http://edamontology.org/format_3006',
		PSL: 'http://edamontology.org/format_3007',
		MAF: 'http://edamontology.org/format_3008',
		'2bit': 'http://edamontology.org/format_3009',
		'.nib': 'http://edamontology.org/format_3010',
		genePred: 'http://edamontology.org/format_3011',
		pgSnp: 'http://edamontology.org/format_3012',
		axt: 'http://edamontology.org/format_3013',
		LAV: 'http://edamontology.org/format_3014',
		Pileup: 'http://edamontology.org/format_3015',
		VCF: 'http://edamontology.org/format_3016',
		SRF: 'http://edamontology.org/format_3017',
		ZTR: 'http://edamontology.org/format_3018',
		GVF: 'http://edamontology.org/format_3019',
		BCF: 'http://edamontology.org/format_3020',
		'Matrix format': 'http://edamontology.org/format_3033',
		'Protein domain classification format':
			'http://edamontology.org/format_3097',
		'Raw SCOP domain classification format':
			'http://edamontology.org/format_3098',
		'Raw CATH domain classification format':
			'http://edamontology.org/format_3099',
		'CATH domain report format': 'http://edamontology.org/format_3100',
		SBRML: 'http://edamontology.org/format_3155',
		BioPAX: 'http://edamontology.org/format_3156',
		'EBI Application Result XML': 'http://edamontology.org/format_3157',
		'PSI MI XML (MIF)': 'http://edamontology.org/format_3158',
		phyloXML: 'http://edamontology.org/format_3159',
		NeXML: 'http://edamontology.org/format_3160',
		'MAGE-ML': 'http://edamontology.org/format_3161',
		'MAGE-TAB': 'http://edamontology.org/format_3162',
		GCDML: 'http://edamontology.org/format_3163',
		GTrack: 'http://edamontology.org/format_3164',
		'Biological pathway or network report format':
			'http://edamontology.org/format_3166',
		'Experiment annotation format': 'http://edamontology.org/format_3167',
		'Cytoband format': 'http://edamontology.org/format_3235',
		CopasiML: 'http://edamontology.org/format_3239',
		CellML: 'http://edamontology.org/format_3240',
		'PSI MI TAB (MITAB)': 'http://edamontology.org/format_3242',
		'PSI-PAR': 'http://edamontology.org/format_3243',
		mzML: 'http://edamontology.org/format_3244',
		'Mass spectrometry data format': 'http://edamontology.org/format_3245',
		TraML: 'http://edamontology.org/format_3246',
		mzIdentML: 'http://edamontology.org/format_3247',
		mzQuantML: 'http://edamontology.org/format_3248',
		GelML: 'http://edamontology.org/format_3249',
		spML: 'http://edamontology.org/format_3250',
		'OWL Functional Syntax': 'http://edamontology.org/format_3252',
		'Manchester OWL Syntax': 'http://edamontology.org/format_3253',
		'KRSS2 Syntax': 'http://edamontology.org/format_3254',
		Turtle: 'http://edamontology.org/format_3255',
		'N-Triples': 'http://edamontology.org/format_3256',
		Notation3: 'http://edamontology.org/format_3257',
		'RDF/XML': 'http://edamontology.org/format_3261',
		'OWL/XML': 'http://edamontology.org/format_3262',
		A2M: 'http://edamontology.org/format_3281',
		SFF: 'http://edamontology.org/format_3284',
		MAP: 'http://edamontology.org/format_3285',
		PED: 'http://edamontology.org/format_3286',
		'Individual genetic data format': 'http://edamontology.org/format_3287',
		'PED/MAP': 'http://edamontology.org/format_3288',
		CT: 'http://edamontology.org/format_3309',
		SS: 'http://edamontology.org/format_3310',
		RNAML: 'http://edamontology.org/format_3311',
		GDE: 'http://edamontology.org/format_3312',
		BLC: 'http://edamontology.org/format_3313',
		'Data index format': 'http://edamontology.org/format_3326',
		BAI: 'http://edamontology.org/format_3327',
		HMMER2: 'http://edamontology.org/format_3328',
		HMMER3: 'http://edamontology.org/format_3329',
		PO: 'http://edamontology.org/format_3330',
		'BLAST XML results format': 'http://edamontology.org/format_3331',
		CRAM: 'http://edamontology.org/format_3462',
		JSON: 'http://edamontology.org/format_3464',
		EPS: 'http://edamontology.org/format_3466',
		GIF: 'http://edamontology.org/format_3467',
		xls: 'http://edamontology.org/format_3468',
		TSV: 'http://edamontology.org/format_3475',
		'Gene expression data format': 'http://edamontology.org/format_3476',
		'Cytoscape input file format': 'http://edamontology.org/format_3477',
		ebwt: 'http://edamontology.org/format_3484',
		RSF: 'http://edamontology.org/format_3485',
		'GCG format variant': 'http://edamontology.org/format_3486',
		BSML: 'http://edamontology.org/format_3487',
		ebwtl: 'http://edamontology.org/format_3491',
		'Ensembl variation file format': 'http://edamontology.org/format_3499',
		docx: 'http://edamontology.org/format_3506',
		'Document format': 'http://edamontology.org/format_3507',
		PDF: 'http://edamontology.org/format_3508',
		'Image format': 'http://edamontology.org/format_3547',
		'DICOM format': 'http://edamontology.org/format_3548',
		nii: 'http://edamontology.org/format_3549',
		mhd: 'http://edamontology.org/format_3550',
		nrrd: 'http://edamontology.org/format_3551',
		'R file format': 'http://edamontology.org/format_3554',
		SPSS: 'http://edamontology.org/format_3555',
		MHTML: 'http://edamontology.org/format_3556',
		IDAT: 'http://edamontology.org/format_3578',
		JPG: 'http://edamontology.org/format_3579',
		rcc: 'http://edamontology.org/format_3580',
		arff: 'http://edamontology.org/format_3581',
		afg: 'http://edamontology.org/format_3582',
		bedgraph: 'http://edamontology.org/format_3583',
		bedstrict: 'http://edamontology.org/format_3584',
		bed6: 'http://edamontology.org/format_3585',
		bed12: 'http://edamontology.org/format_3586',
		chrominfo: 'http://edamontology.org/format_3587',
		customtrack: 'http://edamontology.org/format_3588',
		csfasta: 'http://edamontology.org/format_3589',
		HDF5: 'http://edamontology.org/format_3590',
		TIFF: 'http://edamontology.org/format_3591',
		BMP: 'http://edamontology.org/format_3592',
		im: 'http://edamontology.org/format_3593',
		pcd: 'http://edamontology.org/format_3594',
		pcx: 'http://edamontology.org/format_3595',
		ppm: 'http://edamontology.org/format_3596',
		psd: 'http://edamontology.org/format_3597',
		xbm: 'http://edamontology.org/format_3598',
		xpm: 'http://edamontology.org/format_3599',
		rgb: 'http://edamontology.org/format_3600',
		pbm: 'http://edamontology.org/format_3601',
		pgm: 'http://edamontology.org/format_3602',
		PNG: 'http://edamontology.org/format_3603',
		SVG: 'http://edamontology.org/format_3604',
		rast: 'http://edamontology.org/format_3605',
		'Sequence quality report format (text)':
			'http://edamontology.org/format_3606',
		qual: 'http://edamontology.org/format_3607',
		qualsolexa: 'http://edamontology.org/format_3608',
		qualillumina: 'http://edamontology.org/format_3609',
		qualsolid: 'http://edamontology.org/format_3610',
		qual454: 'http://edamontology.org/format_3611',
		'ENCODE peak format': 'http://edamontology.org/format_3612',
		'ENCODE narrow peak format': 'http://edamontology.org/format_3613',
		'ENCODE broad peak format': 'http://edamontology.org/format_3614',
		bgzip: 'http://edamontology.org/format_3615',
		tabix: 'http://edamontology.org/format_3616',
		'Graph format': 'http://edamontology.org/format_3617',
		xgmml: 'http://edamontology.org/format_3618',
		sif: 'http://edamontology.org/format_3619',
		xlsx: 'http://edamontology.org/format_3620',
		'SQLite format': 'http://edamontology.org/format_3621',
		'Gemini SQLite format': 'http://edamontology.org/format_3622',
		'Index format': 'http://edamontology.org/format_3623',
		snpeffdb: 'http://edamontology.org/format_3624',
		MAT: 'http://edamontology.org/format_3626',
		netCDF: 'http://edamontology.org/format_3650',
		MGF: 'http://edamontology.org/format_3651',
		dta: 'http://edamontology.org/format_3652',
		pkl: 'http://edamontology.org/format_3653',
		mzXML: 'http://edamontology.org/format_3654',
		pepXML: 'http://edamontology.org/format_3655',
		GPML: 'http://edamontology.org/format_3657',
		'K-mer countgraph': 'http://edamontology.org/format_3665',
		mzTab: 'http://edamontology.org/format_3681',
		'imzML metadata file': 'http://edamontology.org/format_3682',
		qcML: 'http://edamontology.org/format_3683',
		'PRIDE XML': 'http://edamontology.org/format_3684',
		'SED-ML': 'http://edamontology.org/format_3685',
		'COMBINE OMEX': 'http://edamontology.org/format_3686',
		'ISA-TAB': 'http://edamontology.org/format_3687',
		SBtab: 'http://edamontology.org/format_3688',
		BCML: 'http://edamontology.org/format_3689',
		BDML: 'http://edamontology.org/format_3690',
		BEL: 'http://edamontology.org/format_3691',
		'SBGN-ML': 'http://edamontology.org/format_3692',
		AGP: 'http://edamontology.org/format_3693',
		PS: 'http://edamontology.org/format_3696',
		'SRA format': 'http://edamontology.org/format_3698',
		VDB: 'http://edamontology.org/format_3699',
		'Tabix index file format': 'http://edamontology.org/format_3700',
		'Sequin format': 'http://edamontology.org/format_3701',
		MSF: 'http://edamontology.org/format_3702',
		'Biodiversity data format': 'http://edamontology.org/format_3706',
		'ABCD format': 'http://edamontology.org/format_3708',
		'GCT/Res format': 'http://edamontology.org/format_3709',
		'WIFF format': 'http://edamontology.org/format_3710',
		'X!Tandem XML': 'http://edamontology.org/format_3711',
		'Thermo RAW': 'http://edamontology.org/format_3712',
		'Mascot .dat file': 'http://edamontology.org/format_3713',
		'MaxQuant APL peaklist format': 'http://edamontology.org/format_3714',
		SBOL: 'http://edamontology.org/format_3725',
		PMML: 'http://edamontology.org/format_3726',
		'OME-TIFF': 'http://edamontology.org/format_3727',
		'LocARNA PP': 'http://edamontology.org/format_3728',
		'dbGaP format': 'http://edamontology.org/format_3729',
		'BIOM format': 'http://edamontology.org/format_3746',
		protXML: 'http://edamontology.org/format_3747',
		'Linked data format': 'http://edamontology.org/format_3748',
		'JSON-LD': 'http://edamontology.org/format_3749',
		YAML: 'http://edamontology.org/format_3750',
		DSV: 'http://edamontology.org/format_3751',
		CSV: 'http://edamontology.org/format_3752',
		'SEQUEST .out file': 'http://edamontology.org/format_3758',
		idXML: 'http://edamontology.org/format_3764',
		'KNIME datatable format': 'http://edamontology.org/format_3765',
		'UniProtKB XML': 'http://edamontology.org/format_3770',
		'UniProtKB RDF': 'http://edamontology.org/format_3771',
		'BioJSON (BioXSD)': 'http://edamontology.org/format_3772',
		BioYAML: 'http://edamontology.org/format_3773',
		'BioJSON (Jalview)': 'http://edamontology.org/format_3774',
		GSuite: 'http://edamontology.org/format_3775',
		BTrack: 'http://edamontology.org/format_3776',
		MCPD: 'http://edamontology.org/format_3777',
		'Annotated text format': 'http://edamontology.org/format_3780',
		'PubAnnotation format': 'http://edamontology.org/format_3781',
		BioC: 'http://edamontology.org/format_3782',
		'PubTator format': 'http://edamontology.org/format_3783',
		'Open Annotation format': 'http://edamontology.org/format_3784',
		'BioNLP Shared Task format': 'http://edamontology.org/format_3785',
		'Query language': 'http://edamontology.org/format_3787',
		SQL: 'http://edamontology.org/format_3788',
		XQuery: 'http://edamontology.org/format_3789',
		SPARQL: 'http://edamontology.org/format_3790',
		xsd: 'http://edamontology.org/format_3804',
		XMFA: 'http://edamontology.org/format_3811',
		GEN: 'http://edamontology.org/format_3812',
		'SAMPLE file format': 'http://edamontology.org/format_3813',
		SDF: 'http://edamontology.org/format_3814',
		Molfile: 'http://edamontology.org/format_3815',
		Mol2: 'http://edamontology.org/format_3816',
		latex: 'http://edamontology.org/format_3817',
		'ELAND format': 'http://edamontology.org/format_3818',
		'Relaxed PHYLIP Interleaved': 'http://edamontology.org/format_3819',
		'Relaxed PHYLIP Sequential': 'http://edamontology.org/format_3820',
		VisML: 'http://edamontology.org/format_3821',
		GML: 'http://edamontology.org/format_3822',
		FASTG: 'http://edamontology.org/format_3823',
		'NMR data format': 'http://edamontology.org/format_3824',
		nmrML: 'http://edamontology.org/format_3825',
		proBAM: 'http://edamontology.org/format_3826',
		proBED: 'http://edamontology.org/format_3827',
		'Raw microarray data format': 'http://edamontology.org/format_3828',
		GPR: 'http://edamontology.org/format_3829',
		ARB: 'http://edamontology.org/format_3830',
		consensusXML: 'http://edamontology.org/format_3832',
		featureXML: 'http://edamontology.org/format_3833',
		mzData: 'http://edamontology.org/format_3834',
		'TIDE TXT': 'http://edamontology.org/format_3835',
		'BLAST XML v2 results format': 'http://edamontology.org/format_3836',
		pptx: 'http://edamontology.org/format_3838',
		ibd: 'http://edamontology.org/format_3839',
		'NLP format': 'http://edamontology.org/format_3841',
		BEAST: 'http://edamontology.org/format_3843',
		'Chado-XML': 'http://edamontology.org/format_3844',
		HSAML: 'http://edamontology.org/format_3845',
		'InterProScan XML': 'http://edamontology.org/format_3846',
		KGML: 'http://edamontology.org/format_3847',
		'PubMed XML': 'http://edamontology.org/format_3848',
		MSAML: 'http://edamontology.org/format_3849',
		OrthoXML: 'http://edamontology.org/format_3850',
		PSDML: 'http://edamontology.org/format_3851',
		SeqXML: 'http://edamontology.org/format_3852',
		'UniParc XML': 'http://edamontology.org/format_3853',
		'UniRef XML': 'http://edamontology.org/format_3854',
		CWL: 'http://edamontology.org/format_3857',
		'Waters RAW': 'http://edamontology.org/format_3858',
		'JCAMP-DX': 'http://edamontology.org/format_3859',
		'NLP annotation format': 'http://edamontology.org/format_3862',
		'NLP corpus format': 'http://edamontology.org/format_3863',
		mirGFF3: 'http://edamontology.org/format_3864',
		'RNA annotation format': 'http://edamontology.org/format_3865',
		'Trajectory format': 'http://edamontology.org/format_3866',
		'Trajectory format (binary)': 'http://edamontology.org/format_3867',
		'Trajectory format (text)': 'http://edamontology.org/format_3868',
		HDF: 'http://edamontology.org/format_3873',
		PCAzip: 'http://edamontology.org/format_3874',
		XTC: 'http://edamontology.org/format_3875',
		TNG: 'http://edamontology.org/format_3876',
		XYZ: 'http://edamontology.org/format_3877',
		mdcrd: 'http://edamontology.org/format_3878',
		'Topology format': 'http://edamontology.org/format_3879',
		'GROMACS top': 'http://edamontology.org/format_3880',
		'AMBER top': 'http://edamontology.org/format_3881',
		PSF: 'http://edamontology.org/format_3882',
		'GROMACS itp': 'http://edamontology.org/format_3883',
		'FF parameter format': 'http://edamontology.org/format_3884',
		BinPos: 'http://edamontology.org/format_3885',
		RST: 'http://edamontology.org/format_3886',
		'CHARMM rtf': 'http://edamontology.org/format_3887',
		'AMBER frcmod': 'http://edamontology.org/format_3888',
		'AMBER off': 'http://edamontology.org/format_3889',
		NMReDATA: 'http://edamontology.org/format_3906',
		BpForms: 'http://edamontology.org/format_3909',
		trr: 'http://edamontology.org/format_3910',
		msh: 'http://edamontology.org/format_3911',
		Loom: 'http://edamontology.org/format_3913',
		Zarr: 'http://edamontology.org/format_3915',
		MTX: 'http://edamontology.org/format_3916',
		BcForms: 'http://edamontology.org/format_3951',
		'N-Quads': 'http://edamontology.org/format_3956',
		Vega: 'http://edamontology.org/format_3969',
		'Vega-lite': 'http://edamontology.org/format_3970',
		NeuroML: 'http://edamontology.org/format_3971',
		BNGL: 'http://edamontology.org/format_3972',
		'Docker image format': 'http://edamontology.org/format_3973',
		'GFA 1': 'http://edamontology.org/format_3975',
		'GFA 2': 'http://edamontology.org/format_3976',
		ObjTables: 'http://edamontology.org/format_3977',
		CONTIG: 'http://edamontology.org/format_3978',
		WEGO: 'http://edamontology.org/format_3979',
		RPKM: 'http://edamontology.org/format_3980',
		'TAR format': 'http://edamontology.org/format_3981',
		CHAIN: 'http://edamontology.org/format_3982',
		NET: 'http://edamontology.org/format_3983',
		QMAP: 'http://edamontology.org/format_3984',
		gxformat2: 'http://edamontology.org/format_3985',
		WMV: 'http://edamontology.org/format_3986',
		'ZIP format': 'http://edamontology.org/format_3987',
		LSM: 'http://edamontology.org/format_3988',
		'GZIP format': 'http://edamontology.org/format_3989',
		AVI: 'http://edamontology.org/format_3990',
		TrackDB: 'http://edamontology.org/format_3991',
		'CIGAR format': 'http://edamontology.org/format_3992',
		'Stereolithography format': 'http://edamontology.org/format_3993',
		U3D: 'http://edamontology.org/format_3994',
		'Texture file format': 'http://edamontology.org/format_3995',
		'Python script': 'http://edamontology.org/format_3996',
		'MPEG-4': 'http://edamontology.org/format_3997',
		'Perl script': 'http://edamontology.org/format_3998',
		'R script': 'http://edamontology.org/format_3999',
		'R markdown': 'http://edamontology.org/format_4000',
		'NIFTI format': 'http://edamontology.org/format_4001',
		pickle: 'http://edamontology.org/format_4002',
		'NumPy format': 'http://edamontology.org/format_4003',
		'SimTools repertoire file format': 'http://edamontology.org/format_4004',
		'Configuration file format': 'http://edamontology.org/format_4005',
		'Zstandard format': 'http://edamontology.org/format_4006',
		'MATLAB script': 'http://edamontology.org/format_4007',
		'Obsolete concept (EDAM)':
			'http://www.geneontology.org/formats/oboInOwl#ObsoleteClass',
	},
	datatype: {},
	operation: {
		Operation: 'http://edamontology.org/operation_0004',
		'Query and retrieval': 'http://edamontology.org/operation_0224',
		'Data retrieval (database cross-reference)':
			'http://edamontology.org/operation_0225',
		Annotation: 'http://edamontology.org/operation_0226',
		Indexing: 'http://edamontology.org/operation_0227',
		'Data index analysis': 'http://edamontology.org/operation_0228',
		'Annotation retrieval (sequence)': 'http://edamontology.org/operation_0229',
		'Sequence generation': 'http://edamontology.org/operation_0230',
		'Sequence editing': 'http://edamontology.org/operation_0231',
		'Sequence merging': 'http://edamontology.org/operation_0232',
		'Sequence conversion': 'http://edamontology.org/operation_0233',
		'Sequence complexity calculation': 'http://edamontology.org/operation_0234',
		'Sequence ambiguity calculation': 'http://edamontology.org/operation_0235',
		'Sequence composition calculation':
			'http://edamontology.org/operation_0236',
		'Repeat sequence analysis': 'http://edamontology.org/operation_0237',
		'Sequence motif discovery': 'http://edamontology.org/operation_0238',
		'Sequence motif recognition': 'http://edamontology.org/operation_0239',
		'Sequence motif comparison': 'http://edamontology.org/operation_0240',
		'Transcription regulatory sequence analysis':
			'http://edamontology.org/operation_0241',
		'Conserved transcription regulatory sequence identification':
			'http://edamontology.org/operation_0242',
		'Protein property calculation (from structure)':
			'http://edamontology.org/operation_0243',
		'Simulation analysis': 'http://edamontology.org/operation_0244',
		'Structural motif discovery': 'http://edamontology.org/operation_0245',
		'Protein domain recognition': 'http://edamontology.org/operation_0246',
		'Protein architecture analysis': 'http://edamontology.org/operation_0247',
		'Residue interaction calculation': 'http://edamontology.org/operation_0248',
		'Protein geometry calculation': 'http://edamontology.org/operation_0249',
		'Protein property calculation': 'http://edamontology.org/operation_0250',
		'Peptide immunogenicity prediction':
			'http://edamontology.org/operation_0252',
		'Sequence feature detection': 'http://edamontology.org/operation_0253',
		'Data retrieval (feature table)': 'http://edamontology.org/operation_0254',
		'Feature table query': 'http://edamontology.org/operation_0255',
		'Sequence feature comparison': 'http://edamontology.org/operation_0256',
		'Data retrieval (sequence alignment)':
			'http://edamontology.org/operation_0257',
		'Sequence alignment analysis': 'http://edamontology.org/operation_0258',
		'Sequence alignment comparison': 'http://edamontology.org/operation_0259',
		'Sequence alignment conversion': 'http://edamontology.org/operation_0260',
		'Nucleic acid property processing':
			'http://edamontology.org/operation_0261',
		'Nucleic acid property calculation':
			'http://edamontology.org/operation_0262',
		'Alternative splicing prediction': 'http://edamontology.org/operation_0264',
		'Frameshift detection': 'http://edamontology.org/operation_0265',
		'Vector sequence detection': 'http://edamontology.org/operation_0266',
		'Protein secondary structure prediction':
			'http://edamontology.org/operation_0267',
		'Protein super-secondary structure prediction':
			'http://edamontology.org/operation_0268',
		'Transmembrane protein prediction':
			'http://edamontology.org/operation_0269',
		'Transmembrane protein analysis': 'http://edamontology.org/operation_0270',
		'Structure prediction': 'http://edamontology.org/operation_0271',
		'Residue contact prediction': 'http://edamontology.org/operation_0272',
		'Protein interaction raw data analysis':
			'http://edamontology.org/operation_0273',
		'Protein-protein interaction prediction (from protein sequence)':
			'http://edamontology.org/operation_0274',
		'Protein-protein interaction prediction (from protein structure)':
			'http://edamontology.org/operation_0275',
		'Protein interaction network analysis':
			'http://edamontology.org/operation_0276',
		'Pathway or network comparison': 'http://edamontology.org/operation_0277',
		'RNA secondary structure prediction':
			'http://edamontology.org/operation_0278',
		'Nucleic acid folding analysis': 'http://edamontology.org/operation_0279',
		'Data retrieval (restriction enzyme annotation)':
			'http://edamontology.org/operation_0280',
		'Genetic marker identification': 'http://edamontology.org/operation_0281',
		'Genetic mapping': 'http://edamontology.org/operation_0282',
		'Linkage analysis': 'http://edamontology.org/operation_0283',
		'Codon usage table generation': 'http://edamontology.org/operation_0284',
		'Codon usage table comparison': 'http://edamontology.org/operation_0285',
		'Codon usage analysis': 'http://edamontology.org/operation_0286',
		'Base position variability plotting':
			'http://edamontology.org/operation_0287',
		'Sequence word comparison': 'http://edamontology.org/operation_0288',
		'Sequence distance matrix generation':
			'http://edamontology.org/operation_0289',
		'Sequence redundancy removal': 'http://edamontology.org/operation_0290',
		'Sequence clustering': 'http://edamontology.org/operation_0291',
		'Sequence alignment': 'http://edamontology.org/operation_0292',
		'Hybrid sequence alignment construction':
			'http://edamontology.org/operation_0293',
		'Structure-based sequence alignment':
			'http://edamontology.org/operation_0294',
		'Structure alignment': 'http://edamontology.org/operation_0295',
		'Sequence profile generation': 'http://edamontology.org/operation_0296',
		'3D profile generation': 'http://edamontology.org/operation_0297',
		'Profile-profile alignment': 'http://edamontology.org/operation_0298',
		'3D profile-to-3D profile alignment':
			'http://edamontology.org/operation_0299',
		'Sequence profile alignment': 'http://edamontology.org/operation_0300',
		'Sequence-to-3D-profile alignment':
			'http://edamontology.org/operation_0301',
		'Protein threading': 'http://edamontology.org/operation_0302',
		'Fold recognition': 'http://edamontology.org/operation_0303',
		'Metadata retrieval': 'http://edamontology.org/operation_0304',
		'Literature search': 'http://edamontology.org/operation_0305',
		'Text mining': 'http://edamontology.org/operation_0306',
		'Virtual PCR': 'http://edamontology.org/operation_0307',
		'PCR primer design': 'http://edamontology.org/operation_0308',
		'Microarray probe design': 'http://edamontology.org/operation_0309',
		'Sequence assembly': 'http://edamontology.org/operation_0310',
		'Microarray data standardisation and normalisation':
			'http://edamontology.org/operation_0311',
		'Sequencing-based expression profile data processing':
			'http://edamontology.org/operation_0312',
		'Expression profile clustering': 'http://edamontology.org/operation_0313',
		'Gene expression profiling': 'http://edamontology.org/operation_0314',
		'Expression profile comparison': 'http://edamontology.org/operation_0315',
		'Functional profiling': 'http://edamontology.org/operation_0316',
		'EST and cDNA sequence analysis': 'http://edamontology.org/operation_0317',
		'Structural genomics target selection':
			'http://edamontology.org/operation_0318',
		'Protein secondary structure assignment':
			'http://edamontology.org/operation_0319',
		'Protein structure assignment': 'http://edamontology.org/operation_0320',
		'Protein structure validation': 'http://edamontology.org/operation_0321',
		'Molecular model refinement': 'http://edamontology.org/operation_0322',
		'Phylogenetic inference': 'http://edamontology.org/operation_0323',
		'Phylogenetic analysis': 'http://edamontology.org/operation_0324',
		'Phylogenetic tree comparison': 'http://edamontology.org/operation_0325',
		'Phylogenetic tree editing': 'http://edamontology.org/operation_0326',
		'Phylogenetic footprinting': 'http://edamontology.org/operation_0327',
		'Protein folding simulation': 'http://edamontology.org/operation_0328',
		'Protein folding pathway prediction':
			'http://edamontology.org/operation_0329',
		'Protein SNP mapping': 'http://edamontology.org/operation_0330',
		'Variant effect prediction': 'http://edamontology.org/operation_0331',
		'Immunogen design': 'http://edamontology.org/operation_0332',
		'Zinc finger prediction': 'http://edamontology.org/operation_0333',
		'Enzyme kinetics calculation': 'http://edamontology.org/operation_0334',
		Formatting: 'http://edamontology.org/operation_0335',
		'Format validation': 'http://edamontology.org/operation_0336',
		Visualisation: 'http://edamontology.org/operation_0337',
		'Sequence database search': 'http://edamontology.org/operation_0338',
		'Structure database search': 'http://edamontology.org/operation_0339',
		'Protein secondary database search':
			'http://edamontology.org/operation_0340',
		'Motif database search': 'http://edamontology.org/operation_0341',
		'Sequence profile database search':
			'http://edamontology.org/operation_0342',
		'Transmembrane protein database search':
			'http://edamontology.org/operation_0343',
		'Sequence retrieval (by code)': 'http://edamontology.org/operation_0344',
		'Sequence retrieval (by keyword)': 'http://edamontology.org/operation_0345',
		'Sequence similarity search': 'http://edamontology.org/operation_0346',
		'Sequence database search (by motif or pattern)':
			'http://edamontology.org/operation_0347',
		'Sequence database search (by amino acid composition)':
			'http://edamontology.org/operation_0348',
		'Sequence database search (by property)':
			'http://edamontology.org/operation_0349',
		'Sequence database search (by sequence using word-based methods)':
			'http://edamontology.org/operation_0350',
		'Sequence database search (by sequence using profile-based methods)':
			'http://edamontology.org/operation_0351',
		'Sequence database search (by sequence using local alignment-based methods)':
			'http://edamontology.org/operation_0352',
		'Sequence database search (by sequence using global alignment-based methods)':
			'http://edamontology.org/operation_0353',
		'Sequence database search (by sequence for primer sequences)':
			'http://edamontology.org/operation_0354',
		'Sequence database search (by molecular weight)':
			'http://edamontology.org/operation_0355',
		'Sequence database search (by isoelectric point)':
			'http://edamontology.org/operation_0356',
		'Structure retrieval (by code)': 'http://edamontology.org/operation_0357',
		'Structure retrieval (by keyword)':
			'http://edamontology.org/operation_0358',
		'Structure database search (by sequence)':
			'http://edamontology.org/operation_0359',
		'Structural similarity search': 'http://edamontology.org/operation_0360',
		'Sequence annotation': 'http://edamontology.org/operation_0361',
		'Genome annotation': 'http://edamontology.org/operation_0362',
		'Reverse complement': 'http://edamontology.org/operation_0363',
		'Random sequence generation': 'http://edamontology.org/operation_0364',
		'Restriction digest': 'http://edamontology.org/operation_0365',
		'Protein sequence cleavage': 'http://edamontology.org/operation_0366',
		'Sequence mutation and randomisation':
			'http://edamontology.org/operation_0367',
		'Sequence masking': 'http://edamontology.org/operation_0368',
		'Sequence cutting': 'http://edamontology.org/operation_0369',
		'Restriction site creation': 'http://edamontology.org/operation_0370',
		'DNA translation': 'http://edamontology.org/operation_0371',
		'DNA transcription': 'http://edamontology.org/operation_0372',
		'Sequence composition calculation (nucleic acid)':
			'http://edamontology.org/operation_0377',
		'Sequence composition calculation (protein)':
			'http://edamontology.org/operation_0378',
		'Repeat sequence detection': 'http://edamontology.org/operation_0379',
		'Repeat sequence organisation analysis':
			'http://edamontology.org/operation_0380',
		'Protein hydropathy calculation (from structure)':
			'http://edamontology.org/operation_0383',
		'Accessible surface calculation': 'http://edamontology.org/operation_0384',
		'Protein hydropathy cluster calculation':
			'http://edamontology.org/operation_0385',
		'Protein dipole moment calculation':
			'http://edamontology.org/operation_0386',
		'Molecular surface calculation': 'http://edamontology.org/operation_0387',
		'Protein binding site prediction (from structure)':
			'http://edamontology.org/operation_0388',
		'Protein-nucleic acid interaction analysis':
			'http://edamontology.org/operation_0389',
		'Protein peeling': 'http://edamontology.org/operation_0390',
		'Protein distance matrix calculation':
			'http://edamontology.org/operation_0391',
		'Contact map calculation': 'http://edamontology.org/operation_0392',
		'Residue cluster calculation': 'http://edamontology.org/operation_0393',
		'Hydrogen bond calculation': 'http://edamontology.org/operation_0394',
		'Residue non-canonical interaction detection':
			'http://edamontology.org/operation_0395',
		'Ramachandran plot calculation': 'http://edamontology.org/operation_0396',
		'Ramachandran plot validation': 'http://edamontology.org/operation_0397',
		'Protein molecular weight calculation':
			'http://edamontology.org/operation_0398',
		'Protein extinction coefficient calculation':
			'http://edamontology.org/operation_0399',
		'Protein pKa calculation': 'http://edamontology.org/operation_0400',
		'Protein hydropathy calculation (from sequence)':
			'http://edamontology.org/operation_0401',
		'Protein titration curve plotting':
			'http://edamontology.org/operation_0402',
		'Protein isoelectric point calculation':
			'http://edamontology.org/operation_0403',
		'Protein hydrogen exchange rate calculation':
			'http://edamontology.org/operation_0404',
		'Protein hydrophobic region calculation':
			'http://edamontology.org/operation_0405',
		'Protein aliphatic index calculation':
			'http://edamontology.org/operation_0406',
		'Protein hydrophobic moment plotting':
			'http://edamontology.org/operation_0407',
		'Protein globularity prediction': 'http://edamontology.org/operation_0408',
		'Protein solubility prediction': 'http://edamontology.org/operation_0409',
		'Protein crystallizability prediction':
			'http://edamontology.org/operation_0410',
		'Protein signal peptide detection (eukaryotes)':
			'http://edamontology.org/operation_0411',
		'Protein signal peptide detection (bacteria)':
			'http://edamontology.org/operation_0412',
		'MHC peptide immunogenicity prediction':
			'http://edamontology.org/operation_0413',
		'Protein feature prediction (from sequence)':
			'http://edamontology.org/operation_0414',
		'Nucleic acid feature detection': 'http://edamontology.org/operation_0415',
		'Epitope mapping': 'http://edamontology.org/operation_0416',
		'PTM site prediction': 'http://edamontology.org/operation_0417',
		'Protein signal peptide detection':
			'http://edamontology.org/operation_0418',
		'Protein binding site prediction (from sequence)':
			'http://edamontology.org/operation_0419',
		'Nucleic acids-binding site prediction':
			'http://edamontology.org/operation_0420',
		'Protein folding site prediction': 'http://edamontology.org/operation_0421',
		'Protein cleavage site prediction':
			'http://edamontology.org/operation_0422',
		'Epitope mapping (MHC Class I)': 'http://edamontology.org/operation_0423',
		'Epitope mapping (MHC Class II)': 'http://edamontology.org/operation_0424',
		'Whole gene prediction': 'http://edamontology.org/operation_0425',
		'Gene component prediction': 'http://edamontology.org/operation_0426',
		'Transposon prediction': 'http://edamontology.org/operation_0427',
		'PolyA signal detection': 'http://edamontology.org/operation_0428',
		'Quadruplex formation site detection':
			'http://edamontology.org/operation_0429',
		'CpG island and isochore detection':
			'http://edamontology.org/operation_0430',
		'Restriction site recognition': 'http://edamontology.org/operation_0431',
		'Nucleosome position prediction': 'http://edamontology.org/operation_0432',
		'Splice site prediction': 'http://edamontology.org/operation_0433',
		'Integrated gene prediction': 'http://edamontology.org/operation_0434',
		'Operon prediction': 'http://edamontology.org/operation_0435',
		'Coding region prediction': 'http://edamontology.org/operation_0436',
		'SECIS element prediction': 'http://edamontology.org/operation_0437',
		'Transcriptional regulatory element prediction':
			'http://edamontology.org/operation_0438',
		'Translation initiation site prediction':
			'http://edamontology.org/operation_0439',
		'Promoter prediction': 'http://edamontology.org/operation_0440',
		'cis-regulatory element prediction':
			'http://edamontology.org/operation_0441',
		'Transcriptional regulatory element prediction (RNA-cis)':
			'http://edamontology.org/operation_0442',
		'trans-regulatory element prediction':
			'http://edamontology.org/operation_0443',
		'S/MAR prediction': 'http://edamontology.org/operation_0444',
		'Transcription factor binding site prediction':
			'http://edamontology.org/operation_0445',
		'Exonic splicing enhancer prediction':
			'http://edamontology.org/operation_0446',
		'Sequence alignment validation': 'http://edamontology.org/operation_0447',
		'Sequence alignment analysis (conservation)':
			'http://edamontology.org/operation_0448',
		'Sequence alignment analysis (site correlation)':
			'http://edamontology.org/operation_0449',
		'Chimera detection': 'http://edamontology.org/operation_0450',
		'Recombination detection': 'http://edamontology.org/operation_0451',
		'Indel detection': 'http://edamontology.org/operation_0452',
		'Nucleosome formation potential prediction':
			'http://edamontology.org/operation_0453',
		'Nucleic acid thermodynamic property calculation':
			'http://edamontology.org/operation_0455',
		'Nucleic acid melting profile plotting':
			'http://edamontology.org/operation_0456',
		'Nucleic acid stitch profile plotting':
			'http://edamontology.org/operation_0457',
		'Nucleic acid melting curve plotting':
			'http://edamontology.org/operation_0458',
		'Nucleic acid probability profile plotting':
			'http://edamontology.org/operation_0459',
		'Nucleic acid temperature profile plotting':
			'http://edamontology.org/operation_0460',
		'Nucleic acid curvature calculation':
			'http://edamontology.org/operation_0461',
		'miRNA target prediction': 'http://edamontology.org/operation_0463',
		'tRNA gene prediction': 'http://edamontology.org/operation_0464',
		'siRNA binding specificity prediction':
			'http://edamontology.org/operation_0465',
		'Protein secondary structure prediction (integrated)':
			'http://edamontology.org/operation_0467',
		'Protein secondary structure prediction (helices)':
			'http://edamontology.org/operation_0468',
		'Protein secondary structure prediction (turns)':
			'http://edamontology.org/operation_0469',
		'Protein secondary structure prediction (coils)':
			'http://edamontology.org/operation_0470',
		'Disulfide bond prediction': 'http://edamontology.org/operation_0471',
		'GPCR prediction': 'http://edamontology.org/operation_0472',
		'GPCR analysis': 'http://edamontology.org/operation_0473',
		'Protein structure prediction': 'http://edamontology.org/operation_0474',
		'Nucleic acid structure prediction':
			'http://edamontology.org/operation_0475',
		'Ab initio structure prediction': 'http://edamontology.org/operation_0476',
		'Protein modelling': 'http://edamontology.org/operation_0477',
		'Molecular docking': 'http://edamontology.org/operation_0478',
		'Backbone modelling': 'http://edamontology.org/operation_0479',
		'Side chain modelling': 'http://edamontology.org/operation_0480',
		'Loop modelling': 'http://edamontology.org/operation_0481',
		'Protein-ligand docking': 'http://edamontology.org/operation_0482',
		'RNA inverse folding': 'http://edamontology.org/operation_0483',
		'SNP detection': 'http://edamontology.org/operation_0484',
		'Radiation Hybrid Mapping': 'http://edamontology.org/operation_0485',
		'Functional mapping': 'http://edamontology.org/operation_0486',
		'Haplotype mapping': 'http://edamontology.org/operation_0487',
		'Linkage disequilibrium calculation':
			'http://edamontology.org/operation_0488',
		'Genetic code prediction': 'http://edamontology.org/operation_0489',
		'Dot plot plotting': 'http://edamontology.org/operation_0490',
		'Pairwise sequence alignment': 'http://edamontology.org/operation_0491',
		'Multiple sequence alignment': 'http://edamontology.org/operation_0492',
		'Pairwise sequence alignment generation (local)':
			'http://edamontology.org/operation_0493',
		'Pairwise sequence alignment generation (global)':
			'http://edamontology.org/operation_0494',
		'Local alignment': 'http://edamontology.org/operation_0495',
		'Global alignment': 'http://edamontology.org/operation_0496',
		'Constrained sequence alignment': 'http://edamontology.org/operation_0497',
		'Consensus-based sequence alignment':
			'http://edamontology.org/operation_0498',
		'Tree-based sequence alignment': 'http://edamontology.org/operation_0499',
		'Secondary structure alignment generation':
			'http://edamontology.org/operation_0500',
		'Protein secondary structure alignment generation':
			'http://edamontology.org/operation_0501',
		'RNA secondary structure alignment':
			'http://edamontology.org/operation_0502',
		'Pairwise structure alignment': 'http://edamontology.org/operation_0503',
		'Multiple structure alignment': 'http://edamontology.org/operation_0504',
		'Structure alignment (protein)': 'http://edamontology.org/operation_0505',
		'Structure alignment (RNA)': 'http://edamontology.org/operation_0506',
		'Pairwise structure alignment generation (local)':
			'http://edamontology.org/operation_0507',
		'Pairwise structure alignment generation (global)':
			'http://edamontology.org/operation_0508',
		'Local structure alignment': 'http://edamontology.org/operation_0509',
		'Global structure alignment': 'http://edamontology.org/operation_0510',
		'Profile-profile alignment (pairwise)':
			'http://edamontology.org/operation_0511',
		'Sequence alignment generation (multiple profile)':
			'http://edamontology.org/operation_0512',
		'3D profile-to-3D profile alignment (pairwise)':
			'http://edamontology.org/operation_0513',
		'Structural profile alignment generation (multiple)':
			'http://edamontology.org/operation_0514',
		'Data retrieval (tool metadata)': 'http://edamontology.org/operation_0515',
		'Data retrieval (database metadata)':
			'http://edamontology.org/operation_0516',
		'PCR primer design (for large scale sequencing)':
			'http://edamontology.org/operation_0517',
		'PCR primer design (for genotyping polymorphisms)':
			'http://edamontology.org/operation_0518',
		'PCR primer design (for gene transcription profiling)':
			'http://edamontology.org/operation_0519',
		'PCR primer design (for conserved primers)':
			'http://edamontology.org/operation_0520',
		'PCR primer design (based on gene structure)':
			'http://edamontology.org/operation_0521',
		'PCR primer design (for methylation PCRs)':
			'http://edamontology.org/operation_0522',
		'Mapping assembly': 'http://edamontology.org/operation_0523',
		'De-novo assembly': 'http://edamontology.org/operation_0524',
		'Genome assembly': 'http://edamontology.org/operation_3440',
		'EST assembly': 'http://edamontology.org/operation_0526',
		'Sequence tag mapping': 'http://edamontology.org/operation_0527',
		'SAGE data processing': 'http://edamontology.org/operation_0528',
		'MPSS data processing': 'http://edamontology.org/operation_0529',
		'SBS data processing': 'http://edamontology.org/operation_0530',
		'Heat map generation': 'http://edamontology.org/operation_0531',
		'Gene expression profile analysis':
			'http://edamontology.org/operation_0532',
		'Expression profile pathway mapping':
			'http://edamontology.org/operation_0533',
		'Protein secondary structure assignment (from coordinate data)':
			'http://edamontology.org/operation_0534',
		'Protein secondary structure assignment (from CD data)':
			'http://edamontology.org/operation_0535',
		'Protein structure assignment (from X-ray crystallographic data)':
			'http://edamontology.org/operation_0536',
		'Protein structure assignment (from NMR data)':
			'http://edamontology.org/operation_0537',
		'Phylogenetic inference (data centric)':
			'http://edamontology.org/operation_0538',
		'Phylogenetic inference (method centric)':
			'http://edamontology.org/operation_0539',
		'Phylogenetic inference (from molecular sequences)':
			'http://edamontology.org/operation_0540',
		'Phylogenetic inference (from continuous quantitative characters)':
			'http://edamontology.org/operation_0541',
		'Phylogenetic inference (from gene frequencies)':
			'http://edamontology.org/operation_0542',
		'Phylogenetic inference (from polymorphism data)':
			'http://edamontology.org/operation_0543',
		'Species tree construction': 'http://edamontology.org/operation_0544',
		'Phylogenetic inference (parsimony methods)':
			'http://edamontology.org/operation_0545',
		'Phylogenetic inference (minimum distance methods)':
			'http://edamontology.org/operation_0546',
		'Phylogenetic inference (maximum likelihood and Bayesian methods)':
			'http://edamontology.org/operation_0547',
		'Phylogenetic inference (quartet methods)':
			'http://edamontology.org/operation_0548',
		'Phylogenetic inference (AI methods)':
			'http://edamontology.org/operation_0549',
		'DNA substitution modelling': 'http://edamontology.org/operation_0550',
		'Phylogenetic tree topology analysis':
			'http://edamontology.org/operation_0551',
		'Phylogenetic tree bootstrapping': 'http://edamontology.org/operation_0552',
		'Gene tree construction': 'http://edamontology.org/operation_0553',
		'Allele frequency distribution analysis':
			'http://edamontology.org/operation_0554',
		'Consensus tree construction': 'http://edamontology.org/operation_0555',
		'Phylogenetic sub/super tree construction':
			'http://edamontology.org/operation_0556',
		'Phylogenetic tree distances calculation':
			'http://edamontology.org/operation_0557',
		'Phylogenetic tree annotation': 'http://edamontology.org/operation_0558',
		'Immunogenicity prediction': 'http://edamontology.org/operation_0559',
		'DNA vaccine design': 'http://edamontology.org/operation_0560',
		'Sequence formatting': 'http://edamontology.org/operation_0561',
		'Sequence alignment formatting': 'http://edamontology.org/operation_0562',
		'Codon usage table formatting': 'http://edamontology.org/operation_0563',
		'Sequence visualisation': 'http://edamontology.org/operation_0564',
		'Sequence alignment visualisation':
			'http://edamontology.org/operation_0565',
		'Sequence cluster visualisation': 'http://edamontology.org/operation_0566',
		'Phylogenetic tree visualisation': 'http://edamontology.org/operation_0567',
		'RNA secondary structure visualisation':
			'http://edamontology.org/operation_0568',
		'Protein secondary structure visualisation':
			'http://edamontology.org/operation_0569',
		'Structure visualisation': 'http://edamontology.org/operation_0570',
		'Expression data visualisation': 'http://edamontology.org/operation_0571',
		'Protein interaction network visualisation':
			'http://edamontology.org/operation_0572',
		'Map drawing': 'http://edamontology.org/operation_0573',
		'Sequence motif rendering': 'http://edamontology.org/operation_0574',
		'Restriction map drawing': 'http://edamontology.org/operation_0575',
		'DNA linear map rendering': 'http://edamontology.org/operation_0577',
		'Plasmid map drawing': 'http://edamontology.org/operation_0578',
		'Operon drawing': 'http://edamontology.org/operation_0579',
		'Nucleic acid folding family identification':
			'http://edamontology.org/operation_1768',
		'Nucleic acid folding energy calculation':
			'http://edamontology.org/operation_1769',
		'Annotation retrieval': 'http://edamontology.org/operation_1774',
		'Protein function prediction': 'http://edamontology.org/operation_1777',
		'Protein function comparison': 'http://edamontology.org/operation_1778',
		'Sequence submission': 'http://edamontology.org/operation_1780',
		'Gene regulatory network analysis':
			'http://edamontology.org/operation_1781',
		Parsing: 'http://edamontology.org/operation_1812',
		'Sequence retrieval': 'http://edamontology.org/operation_1813',
		'Structure retrieval': 'http://edamontology.org/operation_1814',
		'Surface rendering': 'http://edamontology.org/operation_1816',
		'Protein atom surface calculation (accessible)':
			'http://edamontology.org/operation_1817',
		'Protein atom surface calculation (accessible molecular)':
			'http://edamontology.org/operation_1818',
		'Protein residue surface calculation (accessible)':
			'http://edamontology.org/operation_1819',
		'Protein residue surface calculation (vacuum accessible)':
			'http://edamontology.org/operation_1820',
		'Protein residue surface calculation (accessible molecular)':
			'http://edamontology.org/operation_1821',
		'Protein residue surface calculation (vacuum molecular)':
			'http://edamontology.org/operation_1822',
		'Protein surface calculation (accessible molecular)':
			'http://edamontology.org/operation_1823',
		'Protein surface calculation (accessible)':
			'http://edamontology.org/operation_1824',
		'Backbone torsion angle calculation':
			'http://edamontology.org/operation_1825',
		'Full torsion angle calculation': 'http://edamontology.org/operation_1826',
		'Cysteine torsion angle calculation':
			'http://edamontology.org/operation_1827',
		'Tau angle calculation': 'http://edamontology.org/operation_1828',
		'Cysteine bridge detection': 'http://edamontology.org/operation_1829',
		'Free cysteine detection': 'http://edamontology.org/operation_1830',
		'Metal-bound cysteine detection': 'http://edamontology.org/operation_1831',
		'Residue contact calculation (residue-nucleic acid)':
			'http://edamontology.org/operation_1832',
		'Protein-metal contact calculation':
			'http://edamontology.org/operation_1834',
		'Residue contact calculation (residue-negative ion)':
			'http://edamontology.org/operation_1835',
		'Residue bump detection': 'http://edamontology.org/operation_1836',
		'Residue symmetry contact calculation':
			'http://edamontology.org/operation_1837',
		'Residue contact calculation (residue-ligand)':
			'http://edamontology.org/operation_1838',
		'Salt bridge calculation': 'http://edamontology.org/operation_1839',
		'Rotamer likelihood prediction': 'http://edamontology.org/operation_1841',
		'Proline mutation value calculation':
			'http://edamontology.org/operation_1842',
		'Residue packing validation': 'http://edamontology.org/operation_1843',
		'Protein geometry validation': 'http://edamontology.org/operation_1844',
		'PDB file sequence retrieval': 'http://edamontology.org/operation_1845',
		'HET group detection': 'http://edamontology.org/operation_1846',
		'DSSP secondary structure assignment':
			'http://edamontology.org/operation_1847',
		'Structure formatting': 'http://edamontology.org/operation_1848',
		'Protein cysteine and disulfide bond assignment':
			'http://edamontology.org/operation_1850',
		'Residue validation': 'http://edamontology.org/operation_1913',
		'Structure retrieval (water)': 'http://edamontology.org/operation_1914',
		'siRNA duplex prediction': 'http://edamontology.org/operation_2008',
		'Sequence alignment refinement': 'http://edamontology.org/operation_2089',
		'Listfile processing': 'http://edamontology.org/operation_2120',
		'Sequence file editing': 'http://edamontology.org/operation_2121',
		'Sequence alignment file processing':
			'http://edamontology.org/operation_2122',
		'Small molecule data processing': 'http://edamontology.org/operation_2123',
		'Data retrieval (ontology annotation)':
			'http://edamontology.org/operation_2222',
		'Data retrieval (ontology concept)':
			'http://edamontology.org/operation_2224',
		'Representative sequence identification':
			'http://edamontology.org/operation_2233',
		'Structure file processing': 'http://edamontology.org/operation_2234',
		'Data retrieval (sequence profile)':
			'http://edamontology.org/operation_2237',
		'Statistical calculation': 'http://edamontology.org/operation_2238',
		'3D-1D scoring matrix generation': 'http://edamontology.org/operation_2239',
		'Transmembrane protein visualisation':
			'http://edamontology.org/operation_2241',
		Demonstration: 'http://edamontology.org/operation_2246',
		'Data retrieval (pathway or network)':
			'http://edamontology.org/operation_2264',
		'Data retrieval (identifier)': 'http://edamontology.org/operation_2265',
		'Nucleic acid density plotting': 'http://edamontology.org/operation_2284',
		'Sequence analysis': 'http://edamontology.org/operation_2403',
		'Sequence motif analysis': 'http://edamontology.org/operation_2404',
		'Protein interaction data processing':
			'http://edamontology.org/operation_2405',
		'Protein structure analysis': 'http://edamontology.org/operation_2406',
		'Annotation processing': 'http://edamontology.org/operation_2407',
		'Sequence feature analysis': 'http://edamontology.org/operation_2408',
		'Data handling': 'http://edamontology.org/operation_2427',
		'Gene expression analysis': 'http://edamontology.org/operation_2410',
		'Structural profile processing': 'http://edamontology.org/operation_2411',
		'Data index processing': 'http://edamontology.org/operation_2412',
		'Sequence profile processing': 'http://edamontology.org/operation_2413',
		'Protein function analysis': 'http://edamontology.org/operation_2414',
		'Protein folding analysis': 'http://edamontology.org/operation_2415',
		'Protein secondary structure analysis':
			'http://edamontology.org/operation_2416',
		'Physicochemical property data processing':
			'http://edamontology.org/operation_2417',
		'Primer and probe design': 'http://edamontology.org/operation_2419',
		'Operation (typed)': 'http://edamontology.org/operation_2420',
		'Database search': 'http://edamontology.org/operation_2421',
		'Data retrieval': 'http://edamontology.org/operation_2422',
		'Prediction and recognition': 'http://edamontology.org/operation_2423',
		Comparison: 'http://edamontology.org/operation_2424',
		'Optimisation and refinement': 'http://edamontology.org/operation_2425',
		'Modelling and simulation': 'http://edamontology.org/operation_2426',
		Validation: 'http://edamontology.org/operation_2428',
		Mapping: 'http://edamontology.org/operation_2429',
		Design: 'http://edamontology.org/operation_2430',
		'Microarray data processing': 'http://edamontology.org/operation_2432',
		'Codon usage table processing': 'http://edamontology.org/operation_2433',
		'Data retrieval (codon usage table)':
			'http://edamontology.org/operation_2434',
		'Gene expression profile processing':
			'http://edamontology.org/operation_2435',
		'Gene-set enrichment analysis': 'http://edamontology.org/operation_2436',
		'Gene regulatory network prediction':
			'http://edamontology.org/operation_2437',
		'Pathway or network processing': 'http://edamontology.org/operation_2438',
		'RNA secondary structure analysis':
			'http://edamontology.org/operation_2439',
		'Structure processing (RNA)': 'http://edamontology.org/operation_2440',
		'RNA structure prediction': 'http://edamontology.org/operation_2441',
		'DNA structure prediction': 'http://edamontology.org/operation_2442',
		'Phylogenetic tree processing': 'http://edamontology.org/operation_2443',
		'Protein secondary structure processing':
			'http://edamontology.org/operation_2444',
		'Protein interaction network processing':
			'http://edamontology.org/operation_2445',
		'Sequence processing': 'http://edamontology.org/operation_2446',
		'Sequence processing (protein)': 'http://edamontology.org/operation_2447',
		'Sequence processing (nucleic acid)':
			'http://edamontology.org/operation_2448',
		'Sequence comparison': 'http://edamontology.org/operation_2451',
		'Sequence cluster processing': 'http://edamontology.org/operation_2452',
		'Feature table processing': 'http://edamontology.org/operation_2453',
		'Gene prediction': 'http://edamontology.org/operation_2454',
		'GPCR classification': 'http://edamontology.org/operation_2456',
		'GPCR coupling selectivity prediction':
			'http://edamontology.org/operation_2457',
		'Structure processing (protein)': 'http://edamontology.org/operation_2459',
		'Protein atom surface calculation':
			'http://edamontology.org/operation_2460',
		'Protein residue surface calculation':
			'http://edamontology.org/operation_2461',
		'Protein surface calculation': 'http://edamontology.org/operation_2462',
		'Sequence alignment processing': 'http://edamontology.org/operation_2463',
		'Protein-protein binding site prediction':
			'http://edamontology.org/operation_2464',
		'Structure processing': 'http://edamontology.org/operation_2465',
		'Map annotation': 'http://edamontology.org/operation_2466',
		'Data retrieval (protein annotation)':
			'http://edamontology.org/operation_2467',
		'Data retrieval (phylogenetic tree)':
			'http://edamontology.org/operation_2468',
		'Data retrieval (protein interaction annotation)':
			'http://edamontology.org/operation_2469',
		'Data retrieval (protein family annotation)':
			'http://edamontology.org/operation_2470',
		'Data retrieval (RNA family annotation)':
			'http://edamontology.org/operation_2471',
		'Data retrieval (gene annotation)':
			'http://edamontology.org/operation_2472',
		'Data retrieval (genotype and phenotype annotation)':
			'http://edamontology.org/operation_2473',
		'Protein architecture comparison': 'http://edamontology.org/operation_2474',
		'Protein architecture recognition':
			'http://edamontology.org/operation_2475',
		'Molecular dynamics': 'http://edamontology.org/operation_2476',
		'Nucleic acid sequence analysis': 'http://edamontology.org/operation_2478',
		'Protein sequence analysis': 'http://edamontology.org/operation_2479',
		'Structure analysis': 'http://edamontology.org/operation_2480',
		'Nucleic acid structure analysis': 'http://edamontology.org/operation_2481',
		'Secondary structure processing': 'http://edamontology.org/operation_2482',
		'Structure comparison': 'http://edamontology.org/operation_2483',
		'Helical wheel drawing': 'http://edamontology.org/operation_2485',
		'Topology diagram drawing': 'http://edamontology.org/operation_2486',
		'Protein structure comparison': 'http://edamontology.org/operation_2487',
		'Protein secondary structure comparison':
			'http://edamontology.org/operation_2488',
		'Subcellular localisation prediction':
			'http://edamontology.org/operation_2489',
		'Residue contact calculation (residue-residue)':
			'http://edamontology.org/operation_2490',
		'Hydrogen bond calculation (inter-residue)':
			'http://edamontology.org/operation_2491',
		'Protein interaction prediction': 'http://edamontology.org/operation_2492',
		'Codon usage data processing': 'http://edamontology.org/operation_2493',
		'Expression analysis': 'http://edamontology.org/operation_2495',
		'Gene regulatory network processing':
			'http://edamontology.org/operation_2496',
		'Pathway or network analysis': 'http://edamontology.org/operation_2497',
		'Sequencing-based expression profile data analysis':
			'http://edamontology.org/operation_2498',
		'Splicing analysis': 'http://edamontology.org/operation_2499',
		'Microarray raw data analysis': 'http://edamontology.org/operation_2500',
		'Nucleic acid analysis': 'http://edamontology.org/operation_2501',
		'Protein analysis': 'http://edamontology.org/operation_2502',
		'Sequence data processing': 'http://edamontology.org/operation_2503',
		'Structural data processing': 'http://edamontology.org/operation_2504',
		'Text processing': 'http://edamontology.org/operation_2505',
		'Protein sequence alignment analysis':
			'http://edamontology.org/operation_2506',
		'Nucleic acid sequence alignment analysis':
			'http://edamontology.org/operation_2507',
		'Nucleic acid sequence comparison':
			'http://edamontology.org/operation_2508',
		'Protein sequence comparison': 'http://edamontology.org/operation_2509',
		'DNA back-translation': 'http://edamontology.org/operation_2510',
		'Sequence editing (nucleic acid)': 'http://edamontology.org/operation_2511',
		'Sequence editing (protein)': 'http://edamontology.org/operation_2512',
		'Sequence generation (nucleic acid)':
			'http://edamontology.org/operation_2513',
		'Sequence generation (protein)': 'http://edamontology.org/operation_2514',
		'Nucleic acid sequence visualisation':
			'http://edamontology.org/operation_2515',
		'Protein sequence visualisation': 'http://edamontology.org/operation_2516',
		'Nucleic acid structure comparison':
			'http://edamontology.org/operation_2518',
		'Structure processing (nucleic acid)':
			'http://edamontology.org/operation_2519',
		'DNA mapping': 'http://edamontology.org/operation_2520',
		'Map data processing': 'http://edamontology.org/operation_2521',
		'Protein hydropathy calculation': 'http://edamontology.org/operation_2574',
		'Binding site prediction': 'http://edamontology.org/operation_2575',
		'Structure clustering': 'http://edamontology.org/operation_2844',
		'Sequence tagged site (STS) mapping':
			'http://edamontology.org/operation_2871',
		Alignment: 'http://edamontology.org/operation_2928',
		'Protein fragment weight comparison':
			'http://edamontology.org/operation_2929',
		'Protein property comparison': 'http://edamontology.org/operation_2930',
		'Secondary structure comparison': 'http://edamontology.org/operation_2931',
		'Hopp and Woods plotting': 'http://edamontology.org/operation_2932',
		'Cluster textual view generation': 'http://edamontology.org/operation_2934',
		'Clustering profile plotting': 'http://edamontology.org/operation_2935',
		'Dendrograph plotting': 'http://edamontology.org/operation_2936',
		'Proximity map plotting': 'http://edamontology.org/operation_2937',
		'Dendrogram visualisation': 'http://edamontology.org/operation_2938',
		'Principal component visualisation':
			'http://edamontology.org/operation_2939',
		'Scatter plot plotting': 'http://edamontology.org/operation_2940',
		'Whole microarray graph plotting': 'http://edamontology.org/operation_2941',
		'Treemap visualisation': 'http://edamontology.org/operation_2942',
		'Box-Whisker plot plotting': 'http://edamontology.org/operation_2943',
		'Physical mapping': 'http://edamontology.org/operation_2944',
		Analysis: 'http://edamontology.org/operation_2945',
		'Alignment analysis': 'http://edamontology.org/operation_2946',
		'Article analysis': 'http://edamontology.org/operation_2947',
		'Molecular interaction analysis': 'http://edamontology.org/operation_2948',
		'Protein-protein interaction analysis':
			'http://edamontology.org/operation_2949',
		'Residue distance calculation': 'http://edamontology.org/operation_2950',
		'Alignment processing': 'http://edamontology.org/operation_2951',
		'Structure alignment processing': 'http://edamontology.org/operation_2952',
		'Codon usage bias calculation': 'http://edamontology.org/operation_2962',
		'Codon usage bias plotting': 'http://edamontology.org/operation_2963',
		'Codon usage fraction calculation':
			'http://edamontology.org/operation_2964',
		Classification: 'http://edamontology.org/operation_2990',
		'Molecular interaction data processing':
			'http://edamontology.org/operation_2993',
		'Sequence classification': 'http://edamontology.org/operation_2995',
		'Structure classification': 'http://edamontology.org/operation_2996',
		'Protein comparison': 'http://edamontology.org/operation_2997',
		'Nucleic acid comparison': 'http://edamontology.org/operation_2998',
		'Prediction and recognition (protein)':
			'http://edamontology.org/operation_3023',
		'Prediction and recognition (nucleic acid)':
			'http://edamontology.org/operation_3024',
		'Structure editing': 'http://edamontology.org/operation_3080',
		'Sequence alignment editing': 'http://edamontology.org/operation_3081',
		'Pathway or network visualisation':
			'http://edamontology.org/operation_3083',
		'Protein function prediction (from sequence)':
			'http://edamontology.org/operation_3084',
		'Protein sequence feature detection':
			'http://edamontology.org/operation_3087',
		'Protein property calculation (from sequence)':
			'http://edamontology.org/operation_3088',
		'Protein feature prediction (from structure)':
			'http://edamontology.org/operation_3090',
		'Protein feature detection': 'http://edamontology.org/operation_3092',
		'Database search (by sequence)': 'http://edamontology.org/operation_3093',
		'Protein interaction network prediction':
			'http://edamontology.org/operation_3094',
		'Nucleic acid design': 'http://edamontology.org/operation_3095',
		Editing: 'http://edamontology.org/operation_3096',
		'Sequence assembly validation': 'http://edamontology.org/operation_3180',
		'Genome alignment': 'http://edamontology.org/operation_3182',
		'Localised reassembly': 'http://edamontology.org/operation_3183',
		'Sequence assembly visualisation': 'http://edamontology.org/operation_3184',
		'Base-calling': 'http://edamontology.org/operation_3185',
		'Bisulfite mapping': 'http://edamontology.org/operation_3186',
		'Sequence contamination filtering':
			'http://edamontology.org/operation_3187',
		'Trim ends': 'http://edamontology.org/operation_3189',
		'Trim vector': 'http://edamontology.org/operation_3190',
		'Trim to reference': 'http://edamontology.org/operation_3191',
		'Sequence trimming': 'http://edamontology.org/operation_3192',
		'Genome feature comparison': 'http://edamontology.org/operation_3194',
		'Sequencing error detection': 'http://edamontology.org/operation_3195',
		Genotyping: 'http://edamontology.org/operation_3196',
		'Genetic variation analysis': 'http://edamontology.org/operation_3197',
		'Read mapping': 'http://edamontology.org/operation_3198',
		'Split read mapping': 'http://edamontology.org/operation_3199',
		'DNA barcoding': 'http://edamontology.org/operation_3200',
		'SNP calling': 'http://edamontology.org/operation_3201',
		'Polymorphism detection': 'http://edamontology.org/operation_3202',
		'Chromatogram visualisation': 'http://edamontology.org/operation_3203',
		'Methylation analysis': 'http://edamontology.org/operation_3204',
		'Methylation calling': 'http://edamontology.org/operation_3919',
		'Whole genome methylation analysis':
			'http://edamontology.org/operation_3206',
		'Gene methylation analysis': 'http://edamontology.org/operation_3207',
		'Genome visualisation': 'http://edamontology.org/operation_3208',
		'Genome comparison': 'http://edamontology.org/operation_3209',
		'Genome indexing': 'http://edamontology.org/operation_3211',
		'Genome indexing (Burrows-Wheeler)':
			'http://edamontology.org/operation_3212',
		'Genome indexing (suffix arrays)': 'http://edamontology.org/operation_3213',
		'Spectral analysis': 'http://edamontology.org/operation_3214',
		'Peak detection': 'http://edamontology.org/operation_3215',
		Scaffolding: 'http://edamontology.org/operation_3216',
		'Scaffold gap completion': 'http://edamontology.org/operation_3217',
		'Sequencing quality control': 'http://edamontology.org/operation_3218',
		'Read pre-processing': 'http://edamontology.org/operation_3219',
		'Species frequency estimation': 'http://edamontology.org/operation_3221',
		'Peak calling': 'http://edamontology.org/operation_3222',
		'Differential gene expression profiling':
			'http://edamontology.org/operation_3223',
		'Gene set testing': 'http://edamontology.org/operation_3224',
		'Variant classification': 'http://edamontology.org/operation_3225',
		'Variant prioritisation': 'http://edamontology.org/operation_3226',
		'Variant calling': 'http://edamontology.org/operation_3227',
		'Structural variation detection': 'http://edamontology.org/operation_3228',
		'Exome assembly': 'http://edamontology.org/operation_3229',
		'Read depth analysis': 'http://edamontology.org/operation_3230',
		'Gene expression QTL analysis': 'http://edamontology.org/operation_3232',
		'Copy number estimation': 'http://edamontology.org/operation_3233',
		'Primer removal': 'http://edamontology.org/operation_3237',
		'Transcriptome assembly': 'http://edamontology.org/operation_3258',
		'Transcriptome assembly (de novo)':
			'http://edamontology.org/operation_3259',
		'Transcriptome assembly (mapping)':
			'http://edamontology.org/operation_3260',
		'Sequence coordinate conversion': 'http://edamontology.org/operation_3267',
		'Document similarity calculation': 'http://edamontology.org/operation_3278',
		'Document clustering': 'http://edamontology.org/operation_3279',
		'Named-entity and concept recognition':
			'http://edamontology.org/operation_3280',
		'ID mapping': 'http://edamontology.org/operation_3282',
		Anonymisation: 'http://edamontology.org/operation_3283',
		'ID retrieval': 'http://edamontology.org/operation_3289',
		'Sequence checksum generation': 'http://edamontology.org/operation_3348',
		'Bibliography generation': 'http://edamontology.org/operation_3349',
		'Protein quaternary structure prediction':
			'http://edamontology.org/operation_3350',
		'Molecular surface analysis': 'http://edamontology.org/operation_3351',
		'Ontology comparison': 'http://edamontology.org/operation_3353',
		'Format detection': 'http://edamontology.org/operation_3357',
		Splitting: 'http://edamontology.org/operation_3359',
		Generation: 'http://edamontology.org/operation_3429',
		'Nucleic acid sequence feature detection':
			'http://edamontology.org/operation_3430',
		Deposition: 'http://edamontology.org/operation_3431',
		Clustering: 'http://edamontology.org/operation_3432',
		Assembly: 'http://edamontology.org/operation_3433',
		Conversion: 'http://edamontology.org/operation_3434',
		'Standardisation and normalisation':
			'http://edamontology.org/operation_3435',
		Aggregation: 'http://edamontology.org/operation_3436',
		'Article comparison': 'http://edamontology.org/operation_3437',
		Calculation: 'http://edamontology.org/operation_3438',
		'Pathway or network prediction': 'http://edamontology.org/operation_3439',
		Plotting: 'http://edamontology.org/operation_3441',
		'Image analysis': 'http://edamontology.org/operation_3443',
		'Diffraction data analysis': 'http://edamontology.org/operation_3445',
		'Cell migration analysis': 'http://edamontology.org/operation_3446',
		'Diffraction data reduction': 'http://edamontology.org/operation_3447',
		'Neurite measurement': 'http://edamontology.org/operation_3450',
		'Diffraction data integration': 'http://edamontology.org/operation_3453',
		Phasing: 'http://edamontology.org/operation_3454',
		'Molecular replacement': 'http://edamontology.org/operation_3455',
		'Rigid body refinement': 'http://edamontology.org/operation_3456',
		'Single particle analysis': 'http://edamontology.org/operation_3457',
		'Single particle alignment and classification':
			'http://edamontology.org/operation_3458',
		'Functional clustering': 'http://edamontology.org/operation_3459',
		'Taxonomic classification': 'http://edamontology.org/operation_3460',
		'Virulence prediction': 'http://edamontology.org/operation_3461',
		'Expression correlation analysis': 'http://edamontology.org/operation_3463',
		Correlation: 'http://edamontology.org/operation_3465',
		'RNA structure covariance model generation':
			'http://edamontology.org/operation_3469',
		'RNA secondary structure prediction (shape-based)':
			'http://edamontology.org/operation_3470',
		'Nucleic acid folding prediction (alignment-based)':
			'http://edamontology.org/operation_3471',
		'k-mer counting': 'http://edamontology.org/operation_3472',
		'Phylogenetic reconstruction': 'http://edamontology.org/operation_3478',
		'Probabilistic data generation': 'http://edamontology.org/operation_3480',
		'Probabilistic sequence generation':
			'http://edamontology.org/operation_3481',
		'Antimicrobial resistance prediction':
			'http://edamontology.org/operation_3482',
		'Enrichment analysis': 'http://edamontology.org/operation_3501',
		'Chemical similarity enrichment': 'http://edamontology.org/operation_3502',
		'Incident curve plotting': 'http://edamontology.org/operation_3503',
		'Variant pattern analysis': 'http://edamontology.org/operation_3504',
		'Mathematical modelling': 'http://edamontology.org/operation_3545',
		'Microscope image visualisation': 'http://edamontology.org/operation_3552',
		'Image annotation': 'http://edamontology.org/operation_3553',
		Imputation: 'http://edamontology.org/operation_3557',
		'Ontology visualisation': 'http://edamontology.org/operation_3559',
		'Maximum occurence analysis': 'http://edamontology.org/operation_3560',
		'Database comparison': 'http://edamontology.org/operation_3561',
		'Network simulation': 'http://edamontology.org/operation_3562',
		'RNA-seq read count analysis': 'http://edamontology.org/operation_3563',
		'Chemical redundancy removal': 'http://edamontology.org/operation_3564',
		'RNA-seq time series data analysis':
			'http://edamontology.org/operation_3565',
		'Simulated gene expression data generation':
			'http://edamontology.org/operation_3566',
		'Relation extraction': 'http://edamontology.org/operation_3625',
		'Mass spectra calibration': 'http://edamontology.org/operation_3627',
		'Chromatographic alignment': 'http://edamontology.org/operation_3628',
		Deisotoping: 'http://edamontology.org/operation_3629',
		'Protein quantification': 'http://edamontology.org/operation_3630',
		'Peptide identification': 'http://edamontology.org/operation_3631',
		'Isotopic distributions calculation':
			'http://edamontology.org/operation_3632',
		'Retention time prediction': 'http://edamontology.org/operation_3633',
		'Label-free quantification': 'http://edamontology.org/operation_3634',
		'Labeled quantification': 'http://edamontology.org/operation_3635',
		'MRM/SRM': 'http://edamontology.org/operation_3636',
		'Spectral counting': 'http://edamontology.org/operation_3637',
		SILAC: 'http://edamontology.org/operation_3638',
		iTRAQ: 'http://edamontology.org/operation_3639',
		'18O labeling': 'http://edamontology.org/operation_3640',
		'TMT-tag': 'http://edamontology.org/operation_3641',
		Dimethyl: 'http://edamontology.org/operation_3642',
		'Tag-based peptide identification':
			'http://edamontology.org/operation_3643',
		'de Novo sequencing': 'http://edamontology.org/operation_3644',
		'PTM identification': 'http://edamontology.org/operation_3645',
		'Peptide database search': 'http://edamontology.org/operation_3646',
		'Blind peptide database search': 'http://edamontology.org/operation_3647',
		'Validation of peptide-spectrum matches':
			'http://edamontology.org/operation_3648',
		'Target-Decoy': 'http://edamontology.org/operation_3649',
		'Statistical inference': 'http://edamontology.org/operation_3658',
		'Regression analysis': 'http://edamontology.org/operation_3659',
		'Metabolic network modelling': 'http://edamontology.org/operation_3660',
		'SNP annotation': 'http://edamontology.org/operation_3661',
		'Ab-initio gene prediction': 'http://edamontology.org/operation_3662',
		'Homology-based gene prediction': 'http://edamontology.org/operation_3663',
		'Statistical modelling': 'http://edamontology.org/operation_3664',
		'Molecular surface comparison': 'http://edamontology.org/operation_3666',
		'Gene functional annotation': 'http://edamontology.org/operation_3672',
		'Variant filtering': 'http://edamontology.org/operation_3675',
		'Differential binding analysis': 'http://edamontology.org/operation_3677',
		'RNA-Seq analysis': 'http://edamontology.org/operation_3680',
		'Mass spectrum visualisation': 'http://edamontology.org/operation_3694',
		Filtering: 'http://edamontology.org/operation_3695',
		'Reference identification': 'http://edamontology.org/operation_3703',
		'Ion counting': 'http://edamontology.org/operation_3704',
		'Isotope-coded protein label': 'http://edamontology.org/operation_3705',
		'Metabolic labeling': 'http://edamontology.org/operation_3715',
		'Cross-assembly': 'http://edamontology.org/operation_3730',
		'Sample comparison': 'http://edamontology.org/operation_3731',
		'Differential protein expression profiling':
			'http://edamontology.org/operation_3741',
		'Differential gene expression analysis':
			'http://edamontology.org/operation_3742',
		'Multiple sample visualisation': 'http://edamontology.org/operation_3744',
		'Ancestral reconstruction': 'http://edamontology.org/operation_3745',
		'PTM localisation': 'http://edamontology.org/operation_3755',
		'Service management': 'http://edamontology.org/operation_3760',
		'Service discovery': 'http://edamontology.org/operation_3761',
		'Service composition': 'http://edamontology.org/operation_3762',
		'Service invocation': 'http://edamontology.org/operation_3763',
		'Weighted correlation network analysis':
			'http://edamontology.org/operation_3766',
		'Protein identification': 'http://edamontology.org/operation_3767',
		'Text annotation': 'http://edamontology.org/operation_3778',
		'Collapsing methods': 'http://edamontology.org/operation_3791',
		'miRNA expression analysis': 'http://edamontology.org/operation_3792',
		'Read summarisation': 'http://edamontology.org/operation_3793',
		'In vitro selection': 'http://edamontology.org/operation_3795',
		Rarefaction: 'http://edamontology.org/operation_3797',
		'Read binning': 'http://edamontology.org/operation_3798',
		Quantification: 'http://edamontology.org/operation_3799',
		'RNA-Seq quantification': 'http://edamontology.org/operation_3800',
		'Spectral library search': 'http://edamontology.org/operation_3801',
		Sorting: 'http://edamontology.org/operation_3802',
		'Natural product identification': 'http://edamontology.org/operation_3803',
		'DMR identification': 'http://edamontology.org/operation_3809',
		'Multilocus sequence typing': 'http://edamontology.org/operation_3840',
		'Spectrum calculation': 'http://edamontology.org/operation_3860',
		'Trajectory visualization': 'http://edamontology.org/operation_3890',
		'Essential dynamics': 'http://edamontology.org/operation_3891',
		'Forcefield parameterisation': 'http://edamontology.org/operation_3893',
		'DNA profiling': 'http://edamontology.org/operation_3894',
		'Active site prediction': 'http://edamontology.org/operation_3896',
		'Ligand-binding site prediction': 'http://edamontology.org/operation_3897',
		'Metal-binding site prediction': 'http://edamontology.org/operation_3898',
		'Protein-protein docking': 'http://edamontology.org/operation_3899',
		'DNA-binding protein prediction': 'http://edamontology.org/operation_3900',
		'RNA-binding protein prediction': 'http://edamontology.org/operation_3901',
		'RNA binding site prediction': 'http://edamontology.org/operation_3902',
		'DNA binding site prediction': 'http://edamontology.org/operation_3903',
		'Protein disorder prediction': 'http://edamontology.org/operation_3904',
		'Information extraction': 'http://edamontology.org/operation_3907',
		'Information retrieval': 'http://edamontology.org/operation_3908',
		'Genome analysis': 'http://edamontology.org/operation_3918',
		'DNA testing': 'http://edamontology.org/operation_3920',
		'Sequence read processing': 'http://edamontology.org/operation_3921',
		'Genome resequencing': 'http://edamontology.org/operation_3923',
		'Network visualisation': 'http://edamontology.org/operation_3925',
		'Pathway visualisation': 'http://edamontology.org/operation_3926',
		'Network analysis': 'http://edamontology.org/operation_3927',
		'Pathway analysis': 'http://edamontology.org/operation_3928',
		'Metabolic pathway prediction': 'http://edamontology.org/operation_3929',
		Chemometrics: 'http://edamontology.org/operation_3931',
		Demultiplexing: 'http://edamontology.org/operation_3933',
		'Dimensionality reduction': 'http://edamontology.org/operation_3935',
		'Feature selection': 'http://edamontology.org/operation_3936',
		'Feature extraction': 'http://edamontology.org/operation_3937',
		'Virtual screening': 'http://edamontology.org/operation_3938',
		'Metabolic engineering': 'http://edamontology.org/operation_3939',
		'Tree dating': 'http://edamontology.org/operation_3942',
		'Ecological modelling': 'http://edamontology.org/operation_3946',
		'Phylogenetic tree reconciliation':
			'http://edamontology.org/operation_3947',
		'Selection detection': 'http://edamontology.org/operation_3950',
		'Principal component analysis': 'http://edamontology.org/operation_3960',
		'Copy number variation detection': 'http://edamontology.org/operation_3961',
		'Deletion detection': 'http://edamontology.org/operation_3962',
		'Duplication detection': 'http://edamontology.org/operation_3963',
		'Complex CNV detection': 'http://edamontology.org/operation_3964',
		'Amplification detection': 'http://edamontology.org/operation_3965',
		'Adhesin prediction': 'http://edamontology.org/operation_3968',
		'Protein design': 'http://edamontology.org/operation_4008',
		'Small molecule design': 'http://edamontology.org/operation_4009',
	},
	topic: {
		Topic: 'http://edamontology.org/topic_0003',
		'Nucleic acids': 'http://edamontology.org/topic_0077',
		Proteins: 'http://edamontology.org/topic_0078',
		Metabolites: 'http://edamontology.org/topic_0079',
		'Sequence analysis': 'http://edamontology.org/topic_0080',
		'Structure analysis': 'http://edamontology.org/topic_0081',
		'Structure prediction': 'http://edamontology.org/topic_0082',
		Alignment: 'http://edamontology.org/topic_0083',
		Phylogeny: 'http://edamontology.org/topic_0084',
		'Functional genomics': 'http://edamontology.org/topic_0085',
		'Ontology and terminology': 'http://edamontology.org/topic_0089',
		'Information retrieval': 'http://edamontology.org/topic_0090',
		Bioinformatics: 'http://edamontology.org/topic_0091',
		'Data visualisation': 'http://edamontology.org/topic_0092',
		'Nucleic acid thermodynamics': 'http://edamontology.org/topic_0094',
		'Nucleic acid structure analysis': 'http://edamontology.org/topic_0097',
		RNA: 'http://edamontology.org/topic_0099',
		'Nucleic acid restriction': 'http://edamontology.org/topic_0100',
		Mapping: 'http://edamontology.org/topic_0102',
		'Genetic codes and codon usage': 'http://edamontology.org/topic_0107',
		'Protein expression': 'http://edamontology.org/topic_0108',
		'Gene finding': 'http://edamontology.org/topic_0109',
		Transcription: 'http://edamontology.org/topic_0110',
		Promoters: 'http://edamontology.org/topic_1312',
		'Nucleic acid folding': 'http://edamontology.org/topic_0112',
		'Gene structure': 'http://edamontology.org/topic_0114',
		Proteomics: 'http://edamontology.org/topic_0121',
		'Structural genomics': 'http://edamontology.org/topic_0122',
		'Protein properties': 'http://edamontology.org/topic_0123',
		'Protein interactions': 'http://edamontology.org/topic_0128',
		'"Protein folding': 'http://edamontology.org/topic_0130',
		'Two-dimensional gel electrophoresis': 'http://edamontology.org/topic_0133',
		'Mass spectrometry': 'http://edamontology.org/topic_0134',
		'Protein microarrays': 'http://edamontology.org/topic_0135',
		'Protein hydropathy': 'http://edamontology.org/topic_0137',
		'Protein targeting and localisation': 'http://edamontology.org/topic_0140',
		'Protein cleavage sites and proteolysis':
			'http://edamontology.org/topic_0141',
		'Protein structure comparison': 'http://edamontology.org/topic_0143',
		'Protein residue interactions': 'http://edamontology.org/topic_0144',
		'Protein-protein interactions': 'http://edamontology.org/topic_3526',
		'Protein-ligand interactions': 'http://edamontology.org/topic_3514',
		'Protein-nucleic acid interactions': 'http://edamontology.org/topic_3525',
		'Protein design': 'http://edamontology.org/topic_0150',
		'G protein-coupled receptors (GPCR)': 'http://edamontology.org/topic_0151',
		Carbohydrates: 'http://edamontology.org/topic_0152',
		Lipids: 'http://edamontology.org/topic_0153',
		'Small molecules': 'http://edamontology.org/topic_0154',
		'Sequence editing': 'http://edamontology.org/topic_0156',
		'"Sequence composition': 'http://edamontology.org/topic_0157',
		'Sequence motifs': 'http://edamontology.org/topic_0158',
		'Sequence comparison': 'http://edamontology.org/topic_0159',
		'"Sequence sites': 'http://edamontology.org/topic_0160',
		'Sequence database search': 'http://edamontology.org/topic_0163',
		'Sequence clustering': 'http://edamontology.org/topic_0164',
		'Protein structural motifs and surfaces':
			'http://edamontology.org/topic_0166',
		'Structural (3D) profiles': 'http://edamontology.org/topic_0167',
		'Protein structure prediction': 'http://edamontology.org/topic_0172',
		'Nucleic acid structure prediction': 'http://edamontology.org/topic_0173',
		'Ab initio structure prediction': 'http://edamontology.org/topic_0174',
		'Homology modelling': 'http://edamontology.org/topic_0175',
		'Molecular dynamics': 'http://edamontology.org/topic_0176',
		'Molecular docking': 'http://edamontology.org/topic_0177',
		'Protein secondary structure prediction':
			'http://edamontology.org/topic_0178',
		'Protein tertiary structure prediction':
			'http://edamontology.org/topic_0179',
		'Protein fold recognition': 'http://edamontology.org/topic_0180',
		'Sequence alignment': 'http://edamontology.org/topic_0182',
		'Structure alignment': 'http://edamontology.org/topic_0183',
		Threading: 'http://edamontology.org/topic_0184',
		'Sequence profiles and HMMs': 'http://edamontology.org/topic_0188',
		'Phylogeny reconstruction': 'http://edamontology.org/topic_0191',
		Phylogenomics: 'http://edamontology.org/topic_0194',
		'Virtual PCR': 'http://edamontology.org/topic_0195',
		'Sequence assembly': 'http://edamontology.org/topic_0196',
		'Genetic variation': 'http://edamontology.org/topic_0199',
		Microarrays: 'http://edamontology.org/topic_0200',
		Pharmacology: 'http://edamontology.org/topic_0202',
		'Gene expression': 'http://edamontology.org/topic_0203',
		'Gene regulation': 'http://edamontology.org/topic_0204',
		Pharmacogenomics: 'http://edamontology.org/topic_0208',
		'Medicinal chemistry': 'http://edamontology.org/topic_0209',
		Fish: 'http://edamontology.org/topic_0210',
		Flies: 'http://edamontology.org/topic_0211',
		'Mice or rats': 'http://edamontology.org/topic_0213',
		Worms: 'http://edamontology.org/topic_0215',
		'Literature analysis': 'http://edamontology.org/topic_0217',
		'Natural language processing': 'http://edamontology.org/topic_0218',
		'"Data submission': 'http://edamontology.org/topic_0219',
		'"Document': 'http://edamontology.org/topic_0220',
		'Sequence annotation': 'http://edamontology.org/topic_0221',
		'Genome annotation': 'http://edamontology.org/topic_0222',
		NMR: 'http://edamontology.org/topic_0593',
		'Sequence classification': 'http://edamontology.org/topic_0594',
		'Protein classification': 'http://edamontology.org/topic_0595',
		'Sequence motif or profile': 'http://edamontology.org/topic_0598',
		'Protein modifications': 'http://edamontology.org/topic_0601',
		'"Molecular interactions': 'http://edamontology.org/topic_0602',
		Informatics: 'http://edamontology.org/topic_0605',
		'Literature data resources': 'http://edamontology.org/topic_0606',
		'Laboratory information management': 'http://edamontology.org/topic_0607',
		'Cell and tissue culture': 'http://edamontology.org/topic_0608',
		Ecology: 'http://edamontology.org/topic_0610',
		'Electron microscopy': 'http://edamontology.org/topic_0611',
		'Cell cycle': 'http://edamontology.org/topic_0612',
		'Peptides and amino acids': 'http://edamontology.org/topic_0613',
		Organelles: 'http://edamontology.org/topic_0616',
		Ribosomes: 'http://edamontology.org/topic_0617',
		Scents: 'http://edamontology.org/topic_0618',
		'Drugs and target structures': 'http://edamontology.org/topic_0620',
		'Model organisms': 'http://edamontology.org/topic_0621',
		Genomics: 'http://edamontology.org/topic_0622',
		'Gene and protein families': 'http://edamontology.org/topic_0623',
		Chromosomes: 'http://edamontology.org/topic_0624',
		'Genotype and phenotype': 'http://edamontology.org/topic_0625',
		'Gene expression and microarray': 'http://edamontology.org/topic_0629',
		'Probes and primers': 'http://edamontology.org/topic_0632',
		Pathology: 'http://edamontology.org/topic_0634',
		'Specific protein resources': 'http://edamontology.org/topic_0635',
		Taxonomy: 'http://edamontology.org/topic_0637',
		'Protein sequence analysis': 'http://edamontology.org/topic_0639',
		'Nucleic acid sequence analysis': 'http://edamontology.org/topic_0640',
		'Repeat sequences': 'http://edamontology.org/topic_0641',
		'Low complexity sequences': 'http://edamontology.org/topic_0642',
		Proteome: 'http://edamontology.org/topic_0644',
		DNA: 'http://edamontology.org/topic_0654',
		'Coding RNA': 'http://edamontology.org/topic_0655',
		'"Functional': 'http://edamontology.org/topic_0659',
		rRNA: 'http://edamontology.org/topic_0660',
		tRNA: 'http://edamontology.org/topic_0663',
		'Protein secondary structure': 'http://edamontology.org/topic_3542',
		'RNA structure': 'http://edamontology.org/topic_0697',
		'Protein tertiary structure': 'http://edamontology.org/topic_0698',
		'Nucleic acid classification': 'http://edamontology.org/topic_0722',
		'Protein families': 'http://edamontology.org/topic_0724',
		'Protein folds and structural domains':
			'http://edamontology.org/topic_0736',
		'Nucleic acid sequence alignment': 'http://edamontology.org/topic_0740',
		'Protein sequence alignment': 'http://edamontology.org/topic_0741',
		'Nucleic acid sites and features': 'http://edamontology.org/topic_0747',
		'Protein sites and features': 'http://edamontology.org/topic_0748',
		'Transcription factors and regulatory sites':
			'http://edamontology.org/topic_0749',
		'Phosphorylation sites': 'http://edamontology.org/topic_0751',
		'Metabolic pathways': 'http://edamontology.org/topic_0753',
		'Signaling pathways': 'http://edamontology.org/topic_0754',
		'Protein and peptide identification': 'http://edamontology.org/topic_0767',
		Workflows: 'http://edamontology.org/topic_0769',
		'Data types and objects': 'http://edamontology.org/topic_0770',
		'Theoretical biology': 'http://edamontology.org/topic_0771',
		Mitochondria: 'http://edamontology.org/topic_0779',
		'Plant biology': 'http://edamontology.org/topic_0780',
		Virology: 'http://edamontology.org/topic_0781',
		Fungi: 'http://edamontology.org/topic_0782',
		Pathogens: 'http://edamontology.org/topic_0783',
		Arabidopsis: 'http://edamontology.org/topic_0786',
		Rice: 'http://edamontology.org/topic_0787',
		'Genetic mapping and linkage': 'http://edamontology.org/topic_0796',
		'Comparative genomics': 'http://edamontology.org/topic_0797',
		'Mobile genetic elements': 'http://edamontology.org/topic_0798',
		'Human disease': 'http://edamontology.org/topic_0803',
		Immunology: 'http://edamontology.org/topic_0804',
		'Membrane and lipoproteins': 'http://edamontology.org/topic_0820',
		Enzymes: 'http://edamontology.org/topic_0821',
		Primers: 'http://edamontology.org/topic_0922',
		'PolyA signal or sites': 'http://edamontology.org/topic_1302',
		'CpG island and isochores': 'http://edamontology.org/topic_1304',
		'Restriction sites': 'http://edamontology.org/topic_1305',
		'Splice sites': 'http://edamontology.org/topic_1307',
		'Matrix/scaffold attachment sites': 'http://edamontology.org/topic_1308',
		Operon: 'http://edamontology.org/topic_1311',
		'Structural biology': 'http://edamontology.org/topic_1317',
		'Protein membrane regions': 'http://edamontology.org/topic_1456',
		'Structure comparison': 'http://edamontology.org/topic_1770',
		'Function analysis': 'http://edamontology.org/topic_1775',
		'Prokaryotes and Archaea': 'http://edamontology.org/topic_1811',
		'Protein databases': 'http://edamontology.org/topic_2225',
		'Structure determination': 'http://edamontology.org/topic_2226',
		'Cell biology': 'http://edamontology.org/topic_2229',
		Classification: 'http://edamontology.org/topic_2230',
		Lipoproteins: 'http://edamontology.org/topic_2232',
		'Phylogeny visualisation': 'http://edamontology.org/topic_2257',
		Cheminformatics: 'http://edamontology.org/topic_2258',
		'Systems biology': 'http://edamontology.org/topic_2259',
		'Statistics and probability': 'http://edamontology.org/topic_2269',
		'Structure database search': 'http://edamontology.org/topic_2271',
		'Molecular modelling': 'http://edamontology.org/topic_2275',
		'Protein function prediction': 'http://edamontology.org/topic_2276',
		SNP: 'http://edamontology.org/topic_2277',
		'Transmembrane protein prediction': 'http://edamontology.org/topic_2278',
		'Nucleic acid structure comparison': 'http://edamontology.org/topic_2280',
		Exons: 'http://edamontology.org/topic_2397',
		'Gene transcription': 'http://edamontology.org/topic_2399',
		'DNA mutation': 'http://edamontology.org/topic_2533',
		Oncology: 'http://edamontology.org/topic_2640',
		'Toxins and targets': 'http://edamontology.org/topic_2661',
		Introns: 'http://edamontology.org/topic_2754',
		'Tool topic': 'http://edamontology.org/topic_2807',
		'Study topic': 'http://edamontology.org/topic_2809',
		Nomenclature: 'http://edamontology.org/topic_2811',
		'Disease genes and proteins': 'http://edamontology.org/topic_2813',
		'Protein structure analysis': 'http://edamontology.org/topic_2814',
		'Human biology': 'http://edamontology.org/topic_2815',
		'Gene resources': 'http://edamontology.org/topic_2816',
		Yeast: 'http://edamontology.org/topic_2817',
		Eukaryotes: 'http://edamontology.org/topic_2818',
		Invertebrates: 'http://edamontology.org/topic_2819',
		Vertebrates: 'http://edamontology.org/topic_2820',
		'Unicellular eukaryotes': 'http://edamontology.org/topic_2821',
		'Protein structure alignment': 'http://edamontology.org/topic_2826',
		'X-ray diffraction': 'http://edamontology.org/topic_2828',
		'"Ontologies': 'http://edamontology.org/topic_2829',
		'Immunoproteins and antigens': 'http://edamontology.org/topic_2830',
		Molecules: 'http://edamontology.org/topic_2839',
		Toxicology: 'http://edamontology.org/topic_2840',
		'High-throughput sequencing': 'http://edamontology.org/topic_2842',
		'Gene regulatory networks': 'http://edamontology.org/topic_2846',
		'Disease (specific)': 'http://edamontology.org/topic_2847',
		VNTR: 'http://edamontology.org/topic_2867',
		Microsatellites: 'http://edamontology.org/topic_2868',
		RFLP: 'http://edamontology.org/topic_2869',
		'DNA polymorphism': 'http://edamontology.org/topic_2885',
		'Nucleic acid design': 'http://edamontology.org/topic_2953',
		'Primer or probe design': 'http://edamontology.org/topic_3032',
		'Structure databases': 'http://edamontology.org/topic_3038',
		'Nucleic acid structure': 'http://edamontology.org/topic_3039',
		'Sequence databases': 'http://edamontology.org/topic_3041',
		'Nucleic acid sequences': 'http://edamontology.org/topic_3042',
		'Protein sequences': 'http://edamontology.org/topic_3043',
		'Protein interaction networks': 'http://edamontology.org/topic_3044',
		'Molecular biology': 'http://edamontology.org/topic_3047',
		Mammals: 'http://edamontology.org/topic_3048',
		Biodiversity: 'http://edamontology.org/topic_3050',
		'Sequence clusters and classification':
			'http://edamontology.org/topic_3052',
		Genetics: 'http://edamontology.org/topic_3053',
		'Quantitative genetics': 'http://edamontology.org/topic_3055',
		'Population genetics': 'http://edamontology.org/topic_3056',
		'Regulatory RNA': 'http://edamontology.org/topic_3060',
		'Documentation and help': 'http://edamontology.org/topic_3061',
		'Genetic organisation': 'http://edamontology.org/topic_3062',
		'Medical informatics': 'http://edamontology.org/topic_3063',
		'Developmental biology': 'http://edamontology.org/topic_3064',
		Embryology: 'http://edamontology.org/topic_3065',
		Anatomy: 'http://edamontology.org/topic_3067',
		'Literature and language': 'http://edamontology.org/topic_3068',
		Biology: 'http://edamontology.org/topic_3070',
		'Biological databases': 'http://edamontology.org/topic_3071',
		'Sequence feature detection': 'http://edamontology.org/topic_3072',
		'Nucleic acid feature detection': 'http://edamontology.org/topic_3073',
		'Protein feature detection': 'http://edamontology.org/topic_3074',
		'Biological system modelling': 'http://edamontology.org/topic_3075',
		'Data acquisition': 'http://edamontology.org/topic_3077',
		'Genes and proteins resources': 'http://edamontology.org/topic_3078',
		'Protein topological domains': 'http://edamontology.org/topic_3118',
		'Protein variants': 'http://edamontology.org/topic_3120',
		'Expression signals': 'http://edamontology.org/topic_3123',
		'DNA binding sites': 'http://edamontology.org/topic_3125',
		'Nucleic acid repeats': 'http://edamontology.org/topic_3126',
		'DNA replication and recombination': 'http://edamontology.org/topic_3127',
		'Signal or transit peptide': 'http://edamontology.org/topic_3135',
		'Sequence tagged sites': 'http://edamontology.org/topic_3139',
		Sequencing: 'http://edamontology.org/topic_3168',
		'ChIP-seq': 'http://edamontology.org/topic_3169',
		'RNA-Seq': 'http://edamontology.org/topic_3170',
		'DNA methylation': 'http://edamontology.org/topic_3171',
		Metabolomics: 'http://edamontology.org/topic_3172',
		Epigenomics: 'http://edamontology.org/topic_3173',
		Metagenomics: 'http://edamontology.org/topic_3174',
		'Structural variation': 'http://edamontology.org/topic_3175',
		'DNA packaging': 'http://edamontology.org/topic_3176',
		'DNA-Seq': 'http://edamontology.org/topic_3177',
		'RNA-Seq alignment': 'http://edamontology.org/topic_3178',
		'ChIP-on-chip': 'http://edamontology.org/topic_3179',
		'Data security': 'http://edamontology.org/topic_3263',
		'Sample collections': 'http://edamontology.org/topic_3277',
		Biochemistry: 'http://edamontology.org/topic_3292',
		Phylogenetics: 'http://edamontology.org/topic_3293',
		Epigenetics: 'http://edamontology.org/topic_3295',
		Biotechnology: 'http://edamontology.org/topic_3297',
		Phenomics: 'http://edamontology.org/topic_3298',
		'Evolutionary biology': 'http://edamontology.org/topic_3299',
		Physiology: 'http://edamontology.org/topic_3300',
		Microbiology: 'http://edamontology.org/topic_3301',
		Parasitology: 'http://edamontology.org/topic_3302',
		Medicine: 'http://edamontology.org/topic_3303',
		Neurobiology: 'http://edamontology.org/topic_3304',
		'Public health and epidemiology': 'http://edamontology.org/topic_3305',
		Biophysics: 'http://edamontology.org/topic_3306',
		'Computational biology': 'http://edamontology.org/topic_3307',
		Transcriptomics: 'http://edamontology.org/topic_3308',
		Chemistry: 'http://edamontology.org/topic_3314',
		Mathematics: 'http://edamontology.org/topic_3315',
		'Computer science': 'http://edamontology.org/topic_3316',
		Physics: 'http://edamontology.org/topic_3318',
		'RNA splicing': 'http://edamontology.org/topic_3320',
		'Molecular genetics': 'http://edamontology.org/topic_3321',
		'Respiratory medicine': 'http://edamontology.org/topic_3322',
		'Metabolic disease': 'http://edamontology.org/topic_3323',
		'Infectious disease': 'http://edamontology.org/topic_3324',
		'Rare diseases': 'http://edamontology.org/topic_3325',
		'Computational chemistry': 'http://edamontology.org/topic_3332',
		Neurology: 'http://edamontology.org/topic_3334',
		Cardiology: 'http://edamontology.org/topic_3335',
		'Drug discovery': 'http://edamontology.org/topic_3336',
		Biobank: 'http://edamontology.org/topic_3337',
		'Mouse clinic': 'http://edamontology.org/topic_3338',
		'Microbial collection': 'http://edamontology.org/topic_3339',
		'Cell culture collection': 'http://edamontology.org/topic_3340',
		'Clone library': 'http://edamontology.org/topic_3341',
		'Translational medicine': 'http://edamontology.org/topic_3342',
		'Compound libraries and screening': 'http://edamontology.org/topic_3343',
		'Biomedical science': 'http://edamontology.org/topic_3344',
		'Data identity and mapping': 'http://edamontology.org/topic_3345',
		'Sequence search': 'http://edamontology.org/topic_3346',
		Biomarkers: 'http://edamontology.org/topic_3360',
		'Laboratory techniques': 'http://edamontology.org/topic_3361',
		'"Data architecture': 'http://edamontology.org/topic_3365',
		'Data integration and warehousing': 'http://edamontology.org/topic_3366',
		Biomaterials: 'http://edamontology.org/topic_3368',
		'Chemical biology': 'http://edamontology.org/topic_3369',
		'Analytical chemistry': 'http://edamontology.org/topic_3370',
		'Synthetic chemistry': 'http://edamontology.org/topic_3371',
		'Software engineering': 'http://edamontology.org/topic_3372',
		'Drug development': 'http://edamontology.org/topic_3373',
		Biotherapeutics: 'http://edamontology.org/topic_3374',
		'Drug metabolism': 'http://edamontology.org/topic_3375',
		'Medicines research and development': 'http://edamontology.org/topic_3376',
		'Safety sciences': 'http://edamontology.org/topic_3377',
		Pharmacovigilance: 'http://edamontology.org/topic_3378',
		'Preclinical and clinical studies': 'http://edamontology.org/topic_3379',
		Imaging: 'http://edamontology.org/topic_3382',
		Bioimaging: 'http://edamontology.org/topic_3383',
		'Medical imaging': 'http://edamontology.org/topic_3384',
		'Light microscopy': 'http://edamontology.org/topic_3385',
		'Laboratory animal science': 'http://edamontology.org/topic_3386',
		'Marine biology': 'http://edamontology.org/topic_3387',
		'Molecular medicine': 'http://edamontology.org/topic_3388',
		'Nutritional science': 'http://edamontology.org/topic_3390',
		Omics: 'http://edamontology.org/topic_3391',
		'Quality affairs': 'http://edamontology.org/topic_3393',
		'Regulatory affairs': 'http://edamontology.org/topic_3394',
		'Regenerative medicine': 'http://edamontology.org/topic_3395',
		'Systems medicine': 'http://edamontology.org/topic_3396',
		'Veterinary medicine': 'http://edamontology.org/topic_3397',
		Bioengineering: 'http://edamontology.org/topic_3398',
		'Geriatric medicine': 'http://edamontology.org/topic_3399',
		'"Allergy': 'http://edamontology.org/topic_3400',
		'Pain medicine': 'http://edamontology.org/topic_3401',
		Anaesthesiology: 'http://edamontology.org/topic_3402',
		'Critical care medicine': 'http://edamontology.org/topic_3403',
		Dermatology: 'http://edamontology.org/topic_3404',
		Dentistry: 'http://edamontology.org/topic_3405',
		'"Ear': 'http://edamontology.org/topic_3406',
		'Endocrinology and metabolism': 'http://edamontology.org/topic_3407',
		Haematology: 'http://edamontology.org/topic_3408',
		Gastroenterology: 'http://edamontology.org/topic_3409',
		'Gender medicine': 'http://edamontology.org/topic_3410',
		'Gynaecology and obstetrics': 'http://edamontology.org/topic_3411',
		'Hepatic and biliary medicine': 'http://edamontology.org/topic_3412',
		'Infectious tropical disease': 'http://edamontology.org/topic_3413',
		'Trauma medicine': 'http://edamontology.org/topic_3414',
		'Medical toxicology': 'http://edamontology.org/topic_3415',
		'Musculoskeletal medicine': 'http://edamontology.org/topic_3416',
		Opthalmology: 'http://edamontology.org/topic_3417',
		Paediatrics: 'http://edamontology.org/topic_3418',
		Psychiatry: 'http://edamontology.org/topic_3419',
		'Reproductive health': 'http://edamontology.org/topic_3420',
		Surgery: 'http://edamontology.org/topic_3421',
		'Urology and nephrology': 'http://edamontology.org/topic_3422',
		'Complementary medicine': 'http://edamontology.org/topic_3423',
		MRI: 'http://edamontology.org/topic_3444',
		'Neutron diffraction': 'http://edamontology.org/topic_3448',
		Tomography: 'http://edamontology.org/topic_3452',
		'Data mining': 'http://edamontology.org/topic_3473',
		'Machine learning': 'http://edamontology.org/topic_3474',
		'Database management': 'http://edamontology.org/topic_3489',
		Zoology: 'http://edamontology.org/topic_3500',
		'"Protein sites': 'http://edamontology.org/topic_3510',
		'"Nucleic acid sites': 'http://edamontology.org/topic_3511',
		'Gene transcripts': 'http://edamontology.org/topic_3512',
		'Protein-drug interactions': 'http://edamontology.org/topic_3515',
		'Genotyping experiment': 'http://edamontology.org/topic_3516',
		'GWAS study': 'http://edamontology.org/topic_3517',
		'Microarray experiment': 'http://edamontology.org/topic_3518',
		'PCR experiment': 'http://edamontology.org/topic_3519',
		'Proteomics experiment': 'http://edamontology.org/topic_3520',
		'2D PAGE experiment': 'http://edamontology.org/topic_3521',
		'Northern blot experiment': 'http://edamontology.org/topic_3522',
		'RNAi experiment': 'http://edamontology.org/topic_3523',
		'Simulation experiment': 'http://edamontology.org/topic_3524',
		'Cellular process pathways': 'http://edamontology.org/topic_3527',
		'Disease pathways': 'http://edamontology.org/topic_3528',
		'Environmental information processing pathways':
			'http://edamontology.org/topic_3529',
		'Genetic information processing pathways':
			'http://edamontology.org/topic_3530',
		'Protein super-secondary structure': 'http://edamontology.org/topic_3531',
		'Protein active sites': 'http://edamontology.org/topic_3533',
		'Protein binding sites': 'http://edamontology.org/topic_3534',
		'Protein-nucleic acid binding sites': 'http://edamontology.org/topic_3535',
		'Protein cleavage sites': 'http://edamontology.org/topic_3536',
		'Protein chemical modifications': 'http://edamontology.org/topic_3537',
		'Protein disordered structure': 'http://edamontology.org/topic_3538',
		'Protein domains': 'http://edamontology.org/topic_3539',
		'Protein key folding sites': 'http://edamontology.org/topic_3540',
		'Protein post-translational modifications':
			'http://edamontology.org/topic_3541',
		'Protein sequence repeats': 'http://edamontology.org/topic_3543',
		'Protein signal peptides': 'http://edamontology.org/topic_3544',
		'Applied mathematics': 'http://edamontology.org/topic_3569',
		'Pure mathematics': 'http://edamontology.org/topic_3570',
		'Data governance': 'http://edamontology.org/topic_3571',
		'Data quality management': 'http://edamontology.org/topic_3572',
		'Freshwater biology': 'http://edamontology.org/topic_3573',
		'Human genetics': 'http://edamontology.org/topic_3574',
		'Tropical medicine': 'http://edamontology.org/topic_3575',
		'Medical biotechnology': 'http://edamontology.org/topic_3576',
		'Personalised medicine': 'http://edamontology.org/topic_3577',
		'Immunoprecipitation experiment': 'http://edamontology.org/topic_3656',
		'Whole genome sequencing': 'http://edamontology.org/topic_3673',
		'Methylated DNA immunoprecipitation': 'http://edamontology.org/topic_3674',
		'Exome sequencing': 'http://edamontology.org/topic_3676',
		'Experimental design and studies': 'http://edamontology.org/topic_3678',
		'Animal study': 'http://edamontology.org/topic_3679',
		'Microbial ecology': 'http://edamontology.org/topic_3697',
		'RNA immunoprecipitation': 'http://edamontology.org/topic_3794',
		'Population genomics': 'http://edamontology.org/topic_3796',
		'Agricultural science': 'http://edamontology.org/topic_3810',
		'Metagenomic sequencing': 'http://edamontology.org/topic_3837',
		'Environmental science': 'http://edamontology.org/topic_3855',
		'Biomolecular simulation': 'http://edamontology.org/topic_3892',
		'Synthetic biology': 'http://edamontology.org/topic_3895',
		'Genetic engineering': 'http://edamontology.org/topic_3912',
		Proteogenomics: 'http://edamontology.org/topic_3922',
		Immunogenetics: 'http://edamontology.org/topic_3930',
		Cytometry: 'http://edamontology.org/topic_3934',
		'Chromosome conformation capture': 'http://edamontology.org/topic_3940',
		Metatranscriptomics: 'http://edamontology.org/topic_3941',
		Paleogenomics: 'http://edamontology.org/topic_3943',
		Cladistics: 'http://edamontology.org/topic_3944',
		'Molecular evolution': 'http://edamontology.org/topic_3945',
		Immunoinformatics: 'http://edamontology.org/topic_3948',
		Echography: 'http://edamontology.org/topic_3954',
		Fluxomics: 'http://edamontology.org/topic_3955',
		'Protein interaction experiment': 'http://edamontology.org/topic_3957',
		'Copy number variation': 'http://edamontology.org/topic_3958',
		Cytogenetics: 'http://edamontology.org/topic_3959',
		Vaccinology: 'http://edamontology.org/topic_3966',
		Immunomics: 'http://edamontology.org/topic_3967',
		Epistasis: 'http://edamontology.org/topic_3974',
	},
};

export default {};
